{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "cd6175ae-23ac-4491-9aa5-9141d8bfe389",
   "metadata": {},
   "outputs": [],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "b15a3ee6-d9f6-492b-a734-321305c6473c",
   "metadata": {},
   "outputs": [],
   "source": [
    "from html_to_markdown import convert_to_markdown\n",
    "from IPython.display import display, Markdown"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "2c86f325-0d14-4057-84b0-6fe4ac6b6e58",
   "metadata": {},
   "outputs": [],
   "source": [
    "from scifeeder.utils import read_papers_csv\n",
    "from scifeeder.types import Paper\n",
    "papers=list(read_papers_csv('suba_j.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "946f010d-ede9-494b-b6a6-1420ea26ba01",
   "metadata": {},
   "outputs": [],
   "source": [
    "from collections import Counter\n",
    "counts = Counter()\n",
    "for p in papers:\n",
    "    if p.issn and p.doi:\n",
    "        counts[p.issn]+=1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "1043e71b-4836-4feb-ad2f-e0e601c3975d",
   "metadata": {},
   "outputs": [],
   "source": [
    "from scifeeder.selenium_cls import Selenium\n",
    "from scifeeder.issn import ISSN_MAP"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1322eb4a-b331-4eb1-80b0-a0a77a0a6aef",
   "metadata": {},
   "outputs": [],
   "source": [
    "paper = None\n",
    "for idx, p1 in enumerate(reversed(papers)):\n",
    "    if p1.issn:\n",
    "        if p1.issn in {\"0253-9772\"}:\n",
    "            continue\n",
    "    if (p1.issn not in DATA) and p1.doi and p1.year > 1980 and counts[p1.issn] > 0:\n",
    "        paper = p1\n",
    "        break\n",
    "print(idx, len(papers), len(counts), len(ISSN_MAP))\n",
    "if paper:\n",
    "    paper.issn in ISSN_MAP"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "114df66b-6077-4f59-8eb1-0ea3353c0e22",
   "metadata": {},
   "outputs": [],
   "source": [
    "paper = papers[311]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "170793e3-5716-4a5d-bcc1-b01bd895e608",
   "metadata": {},
   "outputs": [],
   "source": [
    "f\"https://doi.org/{paper.doi}\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "22fd0719-91a7-4d16-9e14-f0d6c70fbc47",
   "metadata": {},
   "outputs": [],
   "source": [
    "paper"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0ada6761-a742-45ac-8d27-3a367d2349e5",
   "metadata": {},
   "outputs": [],
   "source": [
    "def toline(p):\n",
    "    return f'\"{p.issn}\": {p.key},'\n",
    "print(toline(paper))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ece02376-dda8-4fc8-b2be-2001dec856de",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "from scifeeder.selenium_cls import Selenium, Soup, StealthSelenium\n",
    "\n",
    "from scifeeder.issn import ISSN_MAP\n",
    "\n",
    "assert paper.issn in ISSN_MAP and paper.doi, paper\n",
    "# web = StealthSelenium(timeout=15, headless=True)\n",
    "web = Selenium(timeout=15, headless=False)\n",
    "markdown, soup = web.run(paper.doi, ISSN_MAP[paper.issn])\n",
    "if markdown:\n",
    "    with open('saved.md', 'wt') as fp:\n",
    "        fp.write(markdown)\n",
    "Markdown(markdown)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "454528c7-5efa-44a7-b8fc-b0b18c97826a",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "assert paper.issn in DATA and paper.doi, paper\n",
    "markdown, soup = web.run(paper.doi, DATA[paper.issn])\n",
    "if markdown:\n",
    "    with open('saved.md', 'wt') as fp:\n",
    "        fp.write(markdown)\n",
    "Markdown(markdown)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c99bcf7f-9ac8-4b37-a58b-0fddba16011a",
   "metadata": {},
   "outputs": [],
   "source": [
    "HTML(web.find_html())"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8bbc5f57-93ee-4e9c-bff3-495a165e2100",
   "metadata": {},
   "source": [
    "----\n",
    "\n",
    "## EPMC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "582cd8e3-cabd-46bf-bad4-47f4677df7a2",
   "metadata": {},
   "outputs": [],
   "source": [
    "from bs4 import BeautifulSoup\n",
    "from io import StringIO, BytesIO\n",
    "from html_to_markdown import convert_to_markdown\n",
    "from IPython.display import HTML, Markdown"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "5277061c-5155-4774-af51-4ee2379d08f5",
   "metadata": {},
   "outputs": [],
   "source": [
    "from scifeeder.epmc import EPMC\n",
    "from scifeeder.xml_utils import Events, PMCEvents, PMCTAGS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "437bd159-f5c7-40a6-9938-3622873ad660",
   "metadata": {},
   "outputs": [],
   "source": [
    "from IPython.display import HTML"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "c9ae290b-6c68-46c8-af84-bd6ac956c7ce",
   "metadata": {},
   "outputs": [],
   "source": [
    "DONE= {'PMC5486124', 'PMC5447075', 'PMC5853234', 'PMC5490910', 'PMC5490927', 'PMC5823009', 'PMC5399074',\n",
    "      'PMC5575511', 'PMC5387099', 'PMC5405513', 'PMC5412409', 'PMC5462045', 'PMC5444443', 'PMC5435427'}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "b7226257-b311-451a-b54b-0a789c268609",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Paper(pmid='28338876', year=2017, title='Subcellular localization of Arabidopsis arogenate dehydratases suggests novel and non-enzymatic roles.', doi='10.1093/jxb/erx024', issn='1460-2431', journal='J Exp Bot', pmcid='PMC5444438')"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "for p in reversed(papers):\n",
    "    if p.pmcid and p.pmcid not in DONE and p.year > 2010:\n",
    "        break\n",
    "p"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "408db355-70f5-413e-adef-2c430f931203",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'https://doi.org/10.1093/jxb/erx024'"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f\"https://doi.org/{p.doi}\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "68094481-b7e6-41fe-a161-8035c2614e99",
   "metadata": {},
   "outputs": [],
   "source": [
    "CSS=\"\"\"\n",
    "<style>\n",
    ".email { color:orange; }\n",
    ".title-group { color: blue; font-size: 14pt; }\n",
    "figure { border: solid orange 1px; padding: 0.5em; }\n",
    ".missing { font-size:larger; color: red; }\n",
    ".sc { color: red; font-weight: bold; }\n",
    "</style>\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "da0f8b1b-4ab4-4cb6-97e9-2f106b3226ee",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       ".email { color:orange; }\n",
       ".title-group { color: blue; font-size: 14pt; }\n",
       "figure { border: solid orange 1px; padding: 0.5em; }\n",
       ".missing { font-size:larger; color: red; }\n",
       ".sc { color: red; font-weight: bold; }\n",
       "</style>\n",
       "<article><div class=\"front\"><div class=\"article-meta\"><h1 class=\"title-group\"><i class=\"article-title\">Subcellular localization of Arabidopsis arogenate dehydratases suggests novel and non-enzymatic roles</i></h1><div class=\"abstract\"><h2 class=\"title\">Highlight</h2><p>We show subcellular localization of all six <i class=\"italic\">Arabidopsis thaliana</i> arogenate dehydratases to chloroplasts and stromules. Furthermore, ADT2 and ADT5 have additional localization patterns suggesting non-enzymatic functions and roles as moonlighting proteins.</p></div><div class=\"abstract\"><h2 class=\"title\">Abstract</h2><p>Arogenate dehydratases (ADTs) catalyze the final step in phenylalanine biosynthesis in plants. The <i class=\"italic\">Arabidopsis thaliana</i> genome encodes a family of six ADTs capable of decarboxylating/dehydrating arogenate into phenylalanine. Using cyan fluorescent protein (CFP)-tagged proteins, the subcellular localization patterns of all six <i class=\"italic\">A. thaliana</i> ADTs were investigated in intact <i class=\"italic\">Nicotiana benthamiana</i> and <i class=\"italic\">A. thaliana</i> leaf cells. We show that <i class=\"italic\">A. thaliana</i> ADTs localize to stroma and stromules (stroma-filled tubules) of chloroplasts. This localization pattern is consistent with the enzymatic function of ADTs as many enzymes required for amino acid biosynthesis are primarily localized to chloroplasts, and stromules are thought to increase metabolite transport from chloroplasts to other cellular compartments. Furthermore, we provide evidence that ADTs have additional, non-enzymatic roles. ADT2 localizes in a ring around the equatorial plane of chloroplasts or to a chloroplast pole, which suggests that ADT2 is a component of the chloroplast division machinery. In addition to chloroplasts, ADT5 was also found in nuclei, again suggesting a non-enzymatic role for ADT5. We also show evidence that ADT5 is transported to the nucleus via stromules. We propose that ADT2 and ADT5 are moonlighting proteins that play an enzymatic role in phenylalanine biosynthesis and a second role in chloroplast division or transcriptional regulation, respectively.</p></div></div></div><main class=\"body\"><hr/><section class=\"sec\"><h2 class=\"title\">Introduction</h2><p>Arogenate dehydratases (ADTs; EC 4.2.1.911) catalyze the final step of phenylalanine biosynthesis through decarboxylation/dehydration of arogenate to form the aromatic amino acid phenylalanine (<a class=\"xref\" href=\"#F1\">Fig. 1A</a>; <a class=\"xref\" href=\"#CIT0051\">Laskar <i class=\"italic\">et al.</i>, 2010</a>; <a class=\"xref\" href=\"#CIT0082\">Tzin and Galili, 2010</a>). In plants phenylalanine serves as a precursor not only of proteins but also of many secondary metabolites, including phenylpropanoids (<a class=\"xref\" href=\"#CIT0032\">Herrmann and Weaver, 1999</a>; <a class=\"xref\" href=\"#CIT0044\">Knaggs, 2003</a>; <a class=\"xref\" href=\"#CIT0087\">Vogt, 2010</a>). Phenylpropanoids have diverse functions, including structural support, pigmentation, and scent formation (<a class=\"xref\" href=\"#CIT0087\">Vogt, 2010</a>), indicating the great importance of phenylalanine biosynthesis in plants.</p><figure class=\"fig\" id=\"F1\"><label>Fig. 1.</label><caption><p>Phenylalanine synthesis, arogenate dehydratases, and stromules. (A) Phenylalanine can be synthesized in plants using either the prephenate (top) or the arogenate (bottom) pathway (<a class=\"xref\" href=\"#CIT0005\">Cho <i class=\"italic\">et al.</i>, 2007</a>; <a class=\"xref\" href=\"#CIT0055\">Maeda and Dudareva, 2012</a>). Prephenate is either decarboxylated/dehydrated to phenylpyruvate (PP) by a prephenate dehydratase (PDT) and PP is then transaminated by a phenylpyruvate aminotransferase (PPAT) to phenylalanine. Alternatively the two enzymatic steps are reversed, whereby prephenate is transaminated to arogenate by a prephenate aminotransferase (PAT) and arogenate is then decarboxylated/dehydrated to phenylalanine by an arogenate dehydratase (ADT). (B) <i class=\"italic\">A. thaliana ADT</i> constructs were cloned in different lengths. The full-length (F) sequence represents the entire <i class=\"italic\">ADT</i> ORF while an N-terminal construct only includes the transit peptide (TP). (C) Schematic diagram of a chloroplast showing the formation of stromules. Stromules are stroma-filled protrusions of the outer and inner membrane from chloroplasts. They can differ in length, forming long thread-like extensions or globular structures.</p></caption>graphic: <a target=\"ext-link\" class=\"graphic\" href=\"erx02401\">erx02401</a></figure><p>The <i class=\"italic\">Arabidopsis thaliana</i> genome encodes a small gene family of six <i class=\"italic\">ADT</i> genes, all sharing similar sequences and domain structures (<a class=\"xref\" href=\"#CIT0014\">Ehlting <i class=\"italic\">et al.</i>, 2005</a>; <a class=\"xref\" href=\"#CIT0005\">Cho <i class=\"italic\">et al.</i>, 2007</a>). Families of ADTs are common in plant genomes and have been identified in both monocot and dicot species (<a class=\"xref\" href=\"#CIT0081\">Tuskan <i class=\"italic\">et al.</i>, 2006</a>; <a class=\"xref\" href=\"#CIT0094\">Yamada <i class=\"italic\">et al.</i>, 2008</a>; <a class=\"xref\" href=\"#CIT0056\">Maeda <i class=\"italic\">et al.</i>, 2010</a>, <a class=\"xref\" href=\"#CIT0057\">2011</a>). The presence of multiple isoforms suggests that ADTs might have evolved different properties, and/or that each ADT is either transcriptionally or post-translationally regulated to allow for distinct functional roles. For example, all six <i class=\"italic\">A. thaliana</i> ADTs accept arogenate as a substrate, as their name suggests, but two of the six ADTs also accept prephenate [meaning they can act as ADTs and prephenate dehydratases (PDTs; <a class=\"xref\" href=\"#F1\">Fig. 1A</a>; <a class=\"xref\" href=\"#CIT0005\">Cho <i class=\"italic\">et al.</i>, 2007</a>; <a class=\"xref\" href=\"#CIT0004\">Bross <i class=\"italic\">et al.</i>, 2011</a>)]. Furthermore, ADTs differentially contribute to lignin content and bolting/flowering transition. Analysis of different <i class=\"italic\">A. thaliana</i> ADT knockout mutants has indicated that specific ADTs preferentially contribute to the synthesis of different downstream products of the phenylpropanoid pathway (<a class=\"xref\" href=\"#CIT0011\">Corea <i class=\"italic\">et al.</i>, 2012<i class=\"italic\">b</i></a>). In lines harboring an adt5 knockout, the amount of phenylalanine, and its proportions relative to tyrosine and tryptophan, were lower in stem tissues compared with the wild-type control (<a class=\"xref\" href=\"#CIT0010\">Corea <i class=\"italic\">et al.</i>, 2012<i class=\"italic\">a</i></a>). Together, these data suggest that ADT5 plays a predominant role in phenylalanine biosynthesis for lignin deposition in stems. Also, the <i class=\"italic\">adt1</i>, but not the <i class=\"italic\">adt4</i>, knockout line exhibited a resistant late bolting/flowering phenotype compared with wild-type <i class=\"italic\">A. thaliana</i> under different environmental conditions, which is consistent with a decreased level of <i class=\"italic\">ADT</i> expression after cold treatment in resistant late bolting/flowering <i class=\"italic\">Beta vulgaris altissima</i> (<a class=\"xref\" href=\"#CIT0030\">Hébrard <i class=\"italic\">et al.</i>, 2013</a>). This suggests that ADTs may be functional targets of DNA methylation in the shoot apical meristem during vernalization, and that the accumulation of phenolic compounds may play a role in floral transition.</p><p>Plant ADTs, including all six <i class=\"italic\">A. thaliana</i> isoforms, have three domains (<a class=\"xref\" href=\"#F1\">Fig. 1B</a>), a putative N-terminal transit peptide (TP), an internal catalytic domain, and a C-terminal ACT (aspartokinase–chorismate mutase–TyrA) domain (<a class=\"xref\" href=\"#CIT0005\">Cho <i class=\"italic\">et al.</i>, 2007</a>). Both the catalytic and ACT domains are conserved across plant, bacterial, and fungal ADTs and PDTs, with the catalytic domain decarboxylating/dehydrating prephenate and/or arogenate (<a class=\"xref\" href=\"#CIT0005\">Cho <i class=\"italic\">et al.</i>, 2007</a>; <a class=\"xref\" href=\"#CIT0004\">Bross <i class=\"italic\">et al.</i>, 2011</a>) while the ACT domain is involved in allosteric regulation induced by ligand binding (<a class=\"xref\" href=\"#CIT0078\">Tan <i class=\"italic\">et al.</i>, 2008</a>; <a class=\"xref\" href=\"#CIT0086\">Vivan <i class=\"italic\">et al.</i>, 2008</a>). The N-terminal domain is unique to plant ADTs and is not found in the bacterial or fungal proteins. In <i class=\"italic\">A. thaliana</i> ADTs, this domain is ~100–130 amino acids in length, and are likely to be chloroplast TPs according to sequence prediction programs, which is consistent with phenylalanine biosynthesis occurring in chloroplasts (<a class=\"xref\" href=\"#CIT0041\">Jung <i class=\"italic\">et al.</i>, 1986</a>; <a class=\"xref\" href=\"#CIT0005\">Cho <i class=\"italic\">et al.</i>, 2007</a>; <a class=\"xref\" href=\"#CIT0052\">Li and Chiu, 2010</a>) and a chloroplastic localization identified for ADTs in protoplasts (<a class=\"xref\" href=\"#CIT0071\">Rippert <i class=\"italic\">et al.</i>, 2009</a>).</p><p>Identifying the subcellular localization of proteins can help to define their functional role and has subsequently led to the identification of new unexpected roles (<a class=\"xref\" href=\"#CIT0077\">Sparkes and Brandizzi, 2012</a>). This approach can be particularly helpful when dissecting and differentiating the biological roles of members within protein families (<a class=\"xref\" href=\"#CIT0042\">Karve <i class=\"italic\">et al.</i>, 2008</a>). In this study, we provide evidence that most <i class=\"italic\">A. thaliana</i> ADTs are targeted to the stroma and stromules (stroma-filled tubules; <a class=\"xref\" href=\"#CIT0046\">Köhler and Hanson, 2000</a>) of chloroplasts and we show that this targeting is dependent on the presence of the TP. This subcellular localization is consistent with the enzymatic role of ADTs in phenylalanine biosynthesis and the proposed role of stromules in increasing metabolite transport (<a class=\"xref\" href=\"#CIT0063\">Natesan <i class=\"italic\">et al.</i>, 2005</a>). In addition, we demonstrate that two of the ADTs, ADT2 and ADT5, have additional subcellular localization patterns that suggest novel, non-enzymatic functions.</p></section><hr/><section class=\"sec-materials-methods\"><h2 class=\"title\">Materials and methods</h2><hr/><section class=\"sec\"><h2 class=\"title\">Growth conditions for bacteria and plants</h2><p>\n",
       "<i class=\"italic\">Escherichia coli</i> DH5α and DH10β strains (Invitrogen catalog nos 11319019 and 18290015, respectively) were used for the maintenance and amplification of plasmid DNA. <i class=\"italic\">Agrobacterium tumefaciens</i> strain LBA4404 containing Ti helper plasmid pAL4404 (NCCB accession no. PC2760; <a class=\"xref\" href=\"#CIT0033\">Hoekema <i class=\"italic\">et al.</i>, 1983</a>; <a class=\"xref\" href=\"#CIT0031\">Hellens <i class=\"italic\">et al.</i>, 2000</a>) was used for transformation of <i class=\"italic\">Nicotiana benthamiana</i> and <i class=\"italic\">A. thaliana</i>. <i class=\"italic\">Agrobacterium tumefaciens</i> strain GV3101 (<a class=\"xref\" href=\"#CIT0048\">Koncz and Schell, 1986</a>; <a class=\"xref\" href=\"#CIT0031\">Hellens <i class=\"italic\">et al.</i>, 2000</a>) was used for the transformation of the <i class=\"italic\">dominant negative myosin XI-2</i> (<i class=\"italic\">dnMyoXI-2</i>) and <i class=\"italic\">dnMyoXI-K/GTD</i> constructs (<a class=\"xref\" href=\"#CIT0002\">Avisar <i class=\"italic\">et al.</i>, 2008</a>). <i class=\"italic\">Escherichia coli</i> and <i class=\"italic\">A. tumefaciens</i> were grown at 37°C in LB medium, and at 28°C in YEB medium, respectively (<a class=\"xref\" href=\"#CIT0083\">Vervliet <i class=\"italic\">et al.</i>, 1975</a>; <a class=\"xref\" href=\"#CIT0003\">Bertani, 2004</a>), with media supplemented with appropriate antibiotics.</p><p>Three- to five-week-old <i class=\"italic\">N. benthamiana</i> were used for localization studies of <i class=\"italic\">A. thaliana</i> ADTs and grown in incubators (Conviron) under 16 h light (80–100 µmol m<sup>−2</sup> s<sup>−1</sup>) and 8 h dark, with light and dark temperatures set to 24°C and 22°C, respectively. <i class=\"italic\">Arabidopsis thaliana</i> accession Columbia-0 (Col-0) was grown for 3–4 weeks with the same photoperiod and a light intensity of 150 µmol m<sup>−2</sup> s<sup>−1</sup>. <i class=\"italic\">Arabidopsis thaliana</i> plants grown for transient transformations were watered with a 20 mM <span class=\"sc\">l</span>-ascorbic acid solution.</p><p>The <i class=\"italic\">adt2-1D</i> mutation is an ethyl methanesulfonate-induced point mutation that causes a serine to be replaced by an alanine in the ACT regulatory domain, leading to an enzyme that is unable to respond to phenylalanine-mediated allosteric inhibition (<a class=\"xref\" href=\"#CIT0034\">Huang <i class=\"italic\">et al.</i>, 2010</a>).</p></section><hr/><section class=\"sec\"><h2 class=\"title\">Cloning of ADT–CFP fusion constructs</h2><p>Primers were designed (<a class=\"xref\" href=\"#T1\">Table 1</a>) to amplify full-length <i class=\"italic\">A. thaliana ADT</i> genes (<i class=\"italic\">ADT1</i>, <i class=\"italic\">At1g11790</i>; <i class=\"italic\">ADT2</i>, <i class=\"italic\">At3g07630</i>; <i class=\"italic\">ADT3</i>, <i class=\"italic\">At2g27820</i>; <i class=\"italic\">ADT4</i>, <i class=\"italic\">At3g44720</i>; <i class=\"italic\">ADT5</i>, <i class=\"italic\">At5g22630</i>; and <i class=\"italic\">ADT6</i>, <i class=\"italic\">At1g08250</i>) (<a class=\"xref\" href=\"#CIT0014\">Ehlting <i class=\"italic\">et al.</i>, 2005</a>; <a class=\"xref\" href=\"#CIT0005\">Cho <i class=\"italic\">et al.</i>, 2007</a>) and tested (Lynnon BioSoft, Version 6). Amplified full-length <i class=\"italic\">ADT</i> sequences contain the entire ORF, including the coding sequence for TPs, ADT/PDT catalytic and ACT regulatory domains. In addition, ADT2 was cloned as the TP sequence only (<a class=\"xref\" href=\"#F1\">Fig. 1B</a>). Primers were designed to include restriction enzyme cleavage and docking sites, to allow for directional integration of PCR fragments into the target vector (<a class=\"xref\" href=\"#T1\">Table 1</a>). <i class=\"italic\">ADT</i> sequences were amplified with Platinum <i class=\"italic\">Taq</i> Polymerase High Fidelity (Invitrogen catalog no. 11304011) with previously cloned <i class=\"italic\">ADT</i> sequences as templates (<a class=\"xref\" href=\"#CIT0005\">Cho <i class=\"italic\">et al.</i>, 2007</a>).</p><div class=\"table-wrap\"><label>Table 1.</label><caption><p>List of primer sequences</p></caption><table><thead><tr><th>\n",
       "<strong class=\"bold\">Name</strong> <sup><strong class=\"bold\"><i class=\"italic\">a</i></strong></sup> </th><th>\n",
       "<strong class=\"bold\">Sequence (5'–3'</strong>)<sup><strong class=\"bold\"><i class=\"italic\">b</i></strong></sup></th><th>\n",
       "<strong class=\"bold\">Restriction enzyme recognition sequence</strong> </th></tr></thead><tbody><tr><td>CFP-For</td><td>\n",
       "<i class=\"italic\">AT</i> <strong class=\"bold\">CGGACCG</strong>GTCGCCACC ATGGTGAGCAAGG</td><td>\n",
       "<i class=\"italic\">Cpo</i>I</td></tr><tr><td>CFP-Rev</td><td>\n",
       "<i class=\"italic\">TCA</i> <strong class=\"bold\">TCTAGA</strong>TTACTTGTA CAGCTCGTCC</td><td>\n",
       "<i class=\"italic\">Xba</i>I</td></tr><tr><td>CFP-Seq</td><td>GATCTGAGCTACACATGC</td><td>N/A</td></tr><tr><td>ADT1-F</td><td>\n",
       "<strong class=\"bold\">AAGCTT</strong>ATGGCTCTGAGGTGTTTTC</td><td>\n",
       "<i class=\"italic\">Hin</i>dIII</td></tr><tr><td>ADT1-R</td><td>\n",
       "<strong class=\"bold\">GGATCC</strong>TGTCTGACTAGATCCATTGG</td><td>\n",
       "<i class=\"italic\">Bam</i>HI</td></tr><tr><td>ADT2-F</td><td>\n",
       "<strong class=\"bold\">AAGCTT</strong>ATGGCAATGCACACTGTTCG</td><td>\n",
       "<i class=\"italic\">Hin</i>dIII</td></tr><tr><td>ADT2-S</td><td>\n",
       "<strong class=\"bold\">AAGCTT</strong>ATGCGTGTTGCGT ATCAGGGAGTACG</td><td>\n",
       "<i class=\"italic\">Hin</i>dIII</td></tr><tr><td>ADT2-R</td><td>\n",
       "<strong class=\"bold\">GGATCC</strong>AAGAGCATTGTA GTGTCCACTGG</td><td>\n",
       "<i class=\"italic\">Bam</i>HI</td></tr><tr><td>ADT2-RTP</td><td>\n",
       "<strong class=\"bold\">GGATCC</strong>TTAACGCGGGAGCCATTAG</td><td>\n",
       "<i class=\"italic\">Bam</i>HI</td></tr><tr><td>ADT3-F</td><td>\n",
       "<strong class=\"bold\">GAATTC</strong>ATGAGAACTCTCTTACCTTC</td><td>\n",
       "<i class=\"italic\">Eco</i>RI</td></tr><tr><td>ADT3-R</td><td>\n",
       "<strong class=\"bold\">GGATCC</strong>ATCAATGAAAATGTTGATGACG</td><td>\n",
       "<i class=\"italic\">Bam</i>HI</td></tr><tr><td>ADT4-F</td><td>\n",
       "<strong class=\"bold\">CTCGAG</strong>ATGCAAGCCGCAACGTCG</td><td>\n",
       "<i class=\"italic\">Xho</i>I</td></tr><tr><td>ADT4-R</td><td>\n",
       "<strong class=\"bold\">GGATCC</strong>AATGCTTCTTCT GTGGATGTCATGG</td><td>\n",
       "<i class=\"italic\">Bam</i>HI</td></tr><tr><td>ADT5-F</td><td>\n",
       "<strong class=\"bold\">CTCGAG</strong>ATGCAAACCATTTCGCC</td><td>\n",
       "<i class=\"italic\">Xho</i>I</td></tr><tr><td>ADT5-R</td><td>\n",
       "<strong class=\"bold\">CCCGGG</strong>TTACGTCTTCGCTAG</td><td>\n",
       "<i class=\"italic\">Sma</i>I</td></tr><tr><td>ADT6-F</td><td>\n",
       "<strong class=\"bold\">GAATTC</strong>ATGAAAGCTCTATCATC</td><td>\n",
       "<i class=\"italic\">Eco</i>RI</td></tr><tr><td>ADT6-R</td><td>\n",
       "<strong class=\"bold\">GGATCC</strong>ATCGATGAAGTTGATG</td><td>\n",
       "<i class=\"italic\">Bam</i>HI</td></tr></tbody></table><div class=\"table-wrap-foot\"></div></div><p>The T-DNA binary vector pEZT-NL (Carnegie Cell Imaging Project, <a target=\"uri\" class=\"uri\" href=\"http://deepgreen.stanford.edu\"> http://deepgreen.stanford.edu, last accessed 13 February 2017</a>) was used in conjunction with the pAL4404 Ti-helper plasmid for expression <i class=\"italic\">in planta</i>. <i class=\"italic\">ADT</i> genes expressed from pEZT-NL are under the control of the <i class=\"italic\">Cauliflower mosaic virus</i> (CaMV) <i class=\"italic\">35S</i> promoter and are translated as C-terminally tagged enhanced green fluorescent protein (EGFP) fusions. For co-localization studies, pEZT-NL was modified by replacing <i class=\"italic\">EGFP</i>, flanked by <i class=\"italic\">Cpo</i>I and <i class=\"italic\">Xba</i>I sites, with the cerulean cyan fluorescent protein (CFP)-coding sequence (<a class=\"xref\" href=\"#CIT0009\">Conley <i class=\"italic\">et al.</i>, 2009<i class=\"italic\">b</i></a>). Both PCR-amplified <i class=\"italic\">CFP</i> sequences and the empty pEZT-NL were double-digested with <i class=\"italic\">Cpo</i>I and <i class=\"italic\">Xba</i>I; the resulting fragments were ligated and then transformed into <i class=\"italic\">E. coli</i>. Positive transformants were selected on LB medium containing gentamicin. The resulting vector was renamed pCB. Using the appropriate restriction enzymes, <i class=\"italic\">ADT</i> genes were cloned into pCB, and all resulting pCB-ADT vectors were sequenced to ensure proper fusion and sequence integrity of the <i class=\"italic\">ADT–CFP</i> sequences.</p><p>To clone the native <i class=\"italic\">ADT5</i> promoter (proNat5), 1 kb upstream of the <i class=\"italic\">ADT5</i> start codon was PCR amplified with primers that added a 5' <i class=\"italic\">Mau</i>BI and a 3' <i class=\"italic\">Xho</i>I restriction site. The amplified <i class=\"italic\">Mau</i>BI–<i class=\"italic\">Xho</i>I fragment was used to replace the CaMV <i class=\"italic\">35S</i> promoter in pCB-ADT5, generating the vector proNat5::ADT5:CFP.</p></section><hr/><section class=\"sec\"><h2 class=\"title\">Cloning filamentous temperature sensitive Z2 (FtsZ2)–yellow fluorescent protein (YFP)</h2><p>The Gateway-compatible vector pLIC6 encoding <i class=\"italic\">FtsZ2-1</i> cDNA was obtained from the ABRC (stock number DKLAT2G36250; <a class=\"xref\" href=\"#CIT0066\">Popescu <i class=\"italic\">et al.</i>, 2007</a>). Restriction digest of pLIC6 with <i class=\"italic\">Hin</i>dIII yielded a 2329 bp restriction fragment containing <i class=\"italic\">FtsZ2-1</i> flanked by <i class=\"italic\">attachment</i>B (<i class=\"italic\">att</i>B) sites. This gel-purified fragment was then recombined into pDONR221 (<a class=\"xref\" href=\"#CIT0029\">Hartley <i class=\"italic\">et al.</i>, 2000</a>). The entry clone created was digested with <i class=\"italic\">Ase</i>I and the expected 2608 bp fragment encoding <i class=\"italic\">FtsZ2-1</i> flanked by <i class=\"italic\">att</i>L sites was isolated and recombined into the destination vector pEarleygate101 (<a class=\"xref\" href=\"#CIT0013\">Earley <i class=\"italic\">et al.</i>, 2006</a>), generating an <i class=\"italic\">FtsZ2–YFP</i> fusion construct with expression regulated by the CaMV <i class=\"italic\">35S</i> promoter.</p></section><hr/><section class=\"sec\"><h2 class=\"title\">Bacterial transformations</h2><p>Plasmid DNA was isolated from overnight <i class=\"italic\">E. coli</i> cultures using an alkaline lysis method (modified from <a class=\"xref\" href=\"#CIT0038\">Ish-Horowicz and Burke, 1981</a>; <a class=\"xref\" href=\"#CIT0072\">Sambrook and Russell, 2001</a>) and transformed into electrocompetent <i class=\"italic\">E. coli</i> (ElectroMax DH5α, Invitrogen), or electrocompetent <i class=\"italic\">A. tumefaciens</i> (<a class=\"xref\" href=\"#CIT0090\">Wise <i class=\"italic\">et al.</i>, 2006</a>), using the Gene Pulser II System (Bio-Rad) set to 2.0 kV, 25 µF capacitance, and 200 Ω or 400 Ω resistance, respectively. Immediately following electroporation, <i class=\"italic\">E. coli</i> and <i class=\"italic\">A. tumefaciens</i> cells were incubated for 1 h in non-selective LB medium, before plating on selective medium. Correct insertion of amplicons into plasmid DNA of positive <i class=\"italic\">E. coli</i> transformants was confirmed by restriction enzyme digestion and sequencing of isolated plasmid DNA. pCB-ADTs were transformed into <i class=\"italic\">A. tumefaciens</i> LBA4404 containing Ti-helper plasmid pAL4404 (<a class=\"xref\" href=\"#CIT0033\">Hoekema <i class=\"italic\">et al.</i>, 1983</a>; <a class=\"xref\" href=\"#CIT0031\">Hellens <i class=\"italic\">et al.</i>, 2000</a>).</p></section><hr/><section class=\"sec\"><h2 class=\"title\">Organelle markers</h2><p>To identify stromules, the TP of the small subunit of tobacco RuBisCO fused to YFP (TP-ssRuBisCO–YFP; <a class=\"xref\" href=\"#CIT0065\">Nelson <i class=\"italic\">et al.</i>, 2007</a>) was used in co-localization experiments. This fusion construct is under the control of a CaMV <i class=\"italic\">35S</i> promoter and, after translation, the TP guides the fusion protein to the chloroplast stroma where it can be used to identify stromules (<a class=\"xref\" href=\"#CIT0065\">Nelson <i class=\"italic\">et al.</i>, 2007</a>).</p><p>The T-DNA-containing binary vector pEarleygate301-YFP, encoding <i class=\"italic\">A. thaliana NUCLEOPORIN1</i> fused to <i class=\"italic\">YFP</i> (<i class=\"italic\">NUP1–YFP</i>) is under the expression of its native promoter (<a class=\"xref\" href=\"#CIT0053\">Lu <i class=\"italic\">et al.</i>, 2010</a>) and was used as a nuclear marker.</p><p>\n",
       "<i class=\"italic\">Agrobacterium tumefaciens</i> GV3101 containing pCB302 encoding the dominant negative form of either <i class=\"italic\">N. benthamiana</i> myosin XI-2 or myosin XI-K/GTD was used to inhibit stromule formation (<a class=\"xref\" href=\"#CIT0002\">Avisar <i class=\"italic\">et al.</i>, 2008</a>; <a class=\"xref\" href=\"#CIT0064\">Natesan <i class=\"italic\">et al.</i>, 2009</a>). Each construct encodes the globular tail domain of the corresponding myosin XI. Expression of the dominant negative constructs in pCB302 is regulated <i class=\"italic\">in planta</i> by the nopaline synthase promoter (<a class=\"xref\" href=\"#CIT0093\">Xiang <i class=\"italic\">et al.</i>, 1999</a>; <a class=\"xref\" href=\"#CIT0002\">Avisar <i class=\"italic\">et al.</i>, 2008</a>).</p></section><hr/><section class=\"sec\"><h2 class=\"title\">Agroinfiltration of tobacco leaves</h2><p>\n",
       "<i class=\"italic\">Nicotiana benthamiana</i> and <i class=\"italic\">A. thaliana</i> were transiently transformed by pressure-infiltrating cultures of <i class=\"italic\">A. tumefaciens</i> (<a class=\"xref\" href=\"#CIT0091\">Wroblewski <i class=\"italic\">et al.</i>, 2005</a>; <a class=\"xref\" href=\"#CIT0092\">Wydro <i class=\"italic\">et al.</i>, 2006</a>; <a class=\"xref\" href=\"#CIT0009\">Conley <i class=\"italic\">et al.</i>, 2009<i class=\"italic\">b</i></a>). <i class=\"italic\">Agrobacterium tumefaciens</i> were grown overnight in 3 ml of YEB medium with the appropriate antibiotics. Then 50 μl of the overnight culture was transferred to 50 ml of YEB containing 25 μl of 200 mM acetosyringone and 500 μl of 1 M MES, and grown until cell density reached an OD<sub>600</sub> of 0.5–0.8. Cells were collected by centrifugation, resuspended in Gamborg’s solution to a final OD<sub>600</sub> of 1.0, and incubated for an additional hour prior to infiltration. For co-infiltration, equal volumes of <i class=\"italic\">A. tumefaciens</i> cultures containing different vectors were combined to maintain a final OD<sub>600</sub> of 1.0. The p19 vector encodes a 19 kDa protein from <i class=\"italic\">Tomato bushy stunt virus</i>, which has been shown to enhance transgene expression through suppression of post-transcriptional gene silencing (PTGS; <a class=\"xref\" href=\"#CIT0076\">Silhavy <i class=\"italic\">et al.</i>, 2002</a>; <a class=\"xref\" href=\"#CIT0088\">Voinnet <i class=\"italic\">et al.</i>, 2003</a>) and was added to all transient transformations.</p><p>A minor variation of this protocol was used for co-expression with <i class=\"italic\">TP-ssRuBisCO–YFP</i> as this construct produced a very strong fluorescence signal compared with that of ADT–CFP. Therefore, <i class=\"italic\">A. tumefaciens</i> strains containing <i class=\"italic\">ADT–CFP</i> and p19 constructs (1:1) were infiltrated 1 d before the infiltration of <i class=\"italic\">TP-ssRuBisCO–YFP</i> and p19 (1:1) constructs. In addition, <i class=\"italic\">A. tumefaciens</i> containing the <i class=\"italic\">TP-ssRuBisCO–YFP</i> construct were infiltrated at a lower OD<sub>600</sub> of ~0.5. Hence visualization of subcellular localization was performed 4 days post-infiltration (dpi) with <i class=\"italic\">TP-ssRuBisCO–YFP</i> (which equals 5 dpi, with an <i class=\"italic\">ADT–CFP</i>).</p><p>Transient transformants were assayed 5 dpi using a Leica SP2 confocal laser scanning microscope equipped with a ×63 water immersion objective. The abaxial surface of leaf tissue was viewed to observe the localization pattern of fluorescent proteins in the lower epidermis and in mesophyll cells. CFP and chlorophyll were excited with a blue diode laser (405 nm), and emission was collected from 440 nm to 485 nm and from 630 nm to 690 nm, respectively. YFP was excited with a 514 nm argon laser and its emission was collected from 540 nm to 550 nm. For co-localization experiments, CFP and YFP emissions were collected sequentially to avoid emission crosstalk of the fluorophore pair (<a class=\"xref\" href=\"#CIT0074\">Shaner <i class=\"italic\">et al.</i>, 2005</a>; <a class=\"xref\" href=\"#CIT0009\">Conley <i class=\"italic\">et al.</i>, 2009<i class=\"italic\">b</i></a>). Images were analyzed using Leica Confocal Software (Leica, V2.61) or ImageJ 1.45s (<a class=\"xref\" href=\"#CIT0073\">Schneider <i class=\"italic\">et al.</i>, 2012</a>). Chlorophyll, CFP, and YFP fluorescence was false colored red, cyan, and yellow, respectively.</p></section><hr/><section class=\"sec\"><h2 class=\"title\">Western blots</h2><p>For one protein extract, three leaf disc samples (9 mm) were collected from transiently transformed <i class=\"italic\">N. benthamiana</i> plants at 4 dpi. Total soluble protein (TSP) was extracted and quantified as described by <a class=\"xref\" href=\"#CIT0008\">Conley <i class=\"italic\">et al.</i> (2009<i class=\"italic\">a</i></a>). For each sample, 10 μg of TSP were size separated by 10% SDS–PAGE. The proteins were probed with a primary anti-GFP antibody (Clontech catalog no. 632380; designed to recognize GFP and fluorescent variants including CFP) at a 1:5000 dilution. Subsequently, a secondary goat anti-mouse IgG (H+L) horseradish peroxidase-conjugated antibody (Bio-Rad catalog no. 170-6516) was used at a 1:3000 dilution. CFP fusion proteins were visualized with the enhanced chemiluminescence (ECL) detection kit (GE Healthcare, Mississauga, ON, Canada).</p></section><hr/><section class=\"sec\"><h2 class=\"title\">Measurements and statistics</h2><p>Stromule and chloroplast lengths were determined using the measuring tool from ImageJ 1.45s (<a class=\"xref\" href=\"#CIT0073\">Schneider <i class=\"italic\">et al.</i>, 2012</a>). Chloroplasts were measured in a straight line across their longest axis. Stromules were considered to be any extensions from chloroplasts that were >1 μm in length. Chloroplasts were analyzed for stromules only if they contained detectable TP-ADT2–CFP fluorescence. For non-linear stromules, several linear measurements were taken to account for bends and curves, and subsequently added together to provide a more accurate measurement of stromule length. Nuclear-localized ADT5–CFP was measured as a proportion of total cells exhibiting ADT5–CFP fluorescence. To determine the proportion of cells having ADT5–CFP within the nucleus, cells were analyzed only if ADT5–CFP was present in the cell/chloroplast. The proportion of chloroplasts with stromules, average stromule and chloroplast lengths, and proportion of cells with ADT5 in the nucleus were analyzed using one-way ANOVA (multiple comparisons) on GraphPad Prism 7.0.</p></section></section><hr/><section class=\"sec\"><h2 class=\"title\">Results</h2><hr/><section class=\"sec\"><h2 class=\"title\">ADTs localize to stroma and stromules of chloroplasts</h2><p>To determine the subcellular localization of ADTs, full-length <i class=\"italic\">A. thaliana</i> ADTs were transiently expressed as CFP fusions in <i class=\"italic\">N. benthamiana</i> leaves. <i class=\"italic\">ADT</i> genes were cloned (<a class=\"xref\" href=\"#F1\">Fig. 1B</a>) upstream of the CFP-coding sequence to maintain the fluorescent tag even if the putative TP is cleaved <i class=\"italic\">in planta</i> (<a class=\"xref\" href=\"#CIT0052\">Li and Chiu, 2010</a>). ADTs were tagged with CFP, instead of GFP, to avoid emission spectra overlap with organelle markers tagged with YFP (<a class=\"xref\" href=\"#CIT0074\">Shaner <i class=\"italic\">et al.</i>, 2005</a>; <a class=\"xref\" href=\"#CIT0065\">Nelson <i class=\"italic\">et al.</i>, 2007</a>; <a class=\"xref\" href=\"#CIT0053\">Lu <i class=\"italic\">et al.</i>, 2010</a>). Leaves of 3- to 5-week-old <i class=\"italic\">N. benthamiana</i> leaves were co-transformed with an <i class=\"italic\">A. tumefaciens</i> strain harboring an <i class=\"italic\">ADT–CFP</i> construct, and a strain encoding p19 to enhance recombinant protein expression (<a class=\"xref\" href=\"#CIT0088\">Voinnet <i class=\"italic\">et al.</i>, 2003</a>). For co-localization experiments, leaves were also co-transformed with <i class=\"italic\">A. tumefaciens</i> carrying a plasmid-encoded organelle marker. ADT–CFP expression was confirmed by isolating total protein and performing western blots (Supplementary Fig. S2 at <i class=\"italic\">JXB</i> online).</p><p>As transformation controls, non-infiltrated tissue, pCB without an insert (empty vector; EV), and pCB infiltrated only with p19 were observed by confocal laser scanning microscopy. For all controls, only chlorophyll autofluorescence was detected and no fluorescence was observed in the CFP and YFP channels in the absence of fusion proteins (Supplementary Fig. S1). ADTs localized to the chloroplast stroma, but were also seen within thread-like structures (e.g. the arrow in ADT2) or as globular structures (e.g. arrows in ADT4) near the chloroplast, but the CFP signal did not directly overlap with chlorophyll autofluorescence (<a class=\"xref\" href=\"#F2\">Fig. 2A</a>). The shape and length of these structures were variable, ranging from short and globular to long and narrow protrusions from the chloroplast body. Unlike ADT1–ADT5, ADT6 did not localize to chloroplasts, but was mostly present in the cytosol (<a class=\"xref\" href=\"#F2\">Fig. 2A</a>, bottom panel).</p><figure class=\"fig\" id=\"F2\"><label>Fig. 2.</label><caption><p>Subcellular localization of ADT–FP fusion proteins and co-localization with TP-ssRuBisCO–YFP. (A) ADT–CFP subcellular localization patterns. ADT1–ADT5 localized to stroma and to areas seemingly close to the chloroplast just outside of the autofluorescence signal generated by chlorophyll. They often appear either in thread-like structures (e.g. the arrow in </p></caption>graphic: <a target=\"ext-link\" class=\"graphic\" href=\"erx02402\">erx02402</a></figure><p>We hypothesized that the thread-like and globular structures were stromules (<a class=\"xref\" href=\"#CIT0046\">Köhler and Hanson, 2000</a>). To confirm that ADTs do localize to stromules, the <i class=\"italic\">ADT–CFP</i> fusion constructs were co-expressed with the TP of the small subunit of RuBisCO fused to YFP (TP-ssRuBisCO–YFP). This construct is known to localize to the stroma, and therefore can be used to identify stromules (<a class=\"xref\" href=\"#CIT0065\">Nelson <i class=\"italic\">et al.</i>, 2007</a>). The fluorescence of CFP fusion proteins with ADT1–ADT5 overlapped with the fluorescence of TP-ssRuBisCO–YFP, which confirms that these ADTs are targeted to stromules within the chloroplasts (<a class=\"xref\" href=\"#F2\">Fig. 2B</a>). The fluorescence of ADT6–CFP did not overlap with the fluorescence of TP-ssRuBisCO–YFP as ADT6 is mostly found within the cytosol (<a class=\"xref\" href=\"#F2\">Fig. 2B</a>, bottom panel).</p><p>To determine if the TP domain of ADTs is responsible for chloroplast and stromule localization, the TP sequence was expressed as a TP-ADT2–CFP fusion protein and was detected in chloroplasts and stromules (<a class=\"xref\" href=\"#F3\">Fig. 3</a>). These data are consistent with the TP being sufficient to target ADT sequences to chloroplasts and stromules.</p><figure class=\"fig\" id=\"F3\"><label>Fig. 3.</label><caption><p>Localization of ADTs to the chloroplast is dependent on the transit peptide sequences. To test if the transit peptide sequences are sufficient for the transport of ADTs to the chloroplast, the first 99 amino acids of ADT2 (TP-ADT2–CFP) were expressed transiently in <i class=\"italic\">N. benthamiana</i> leaves. </p></caption>graphic: <a target=\"ext-link\" class=\"graphic\" href=\"erx02403\">erx02403</a></figure></section><hr/><section class=\"sec\"><h2 class=\"title\">ADT2 localizes to chloroplasts in a ring structure</h2><p>ADT2–CFP displayed a unique localization pattern compared with the other ADTs (<a class=\"xref\" href=\"#F4\">Fig. 4A</a>). In chloroplasts with no apparent central constriction, ADT2–CFP localized to a band at the equatorial plane. Stacked confocal images showed that ADT2 formed a ring around the center of the chloroplast (data not shown). In elongated chloroplasts with a slight indentation, suggestive of an early chloroplast division stage, ADT2–CFP localized as a band around the middle of the elongation exactly at the point of indentation. In chloroplasts with a clear indentation, indicative of a later stage of division, ADT2–CFP was found at the site of constriction. ADT2 localization to the poles of chloroplasts was consistent with remnants of the division ring on daughter chloroplasts (<a class=\"xref\" href=\"#CIT0060\">Miyagishima, 2011</a>). These ADT2 localization patterns are strikingly similar to those of proteins that are involved in chloroplast division, a process requiring placement of multiple proteinaceous rings followed by constriction that partitions the chloroplast into two equal-sized daughter chloroplasts (<a class=\"xref\" href=\"#F4\">Fig. 4B</a>; <a class=\"xref\" href=\"#CIT0060\">Miyagishima, 2011</a>).</p><figure class=\"fig\" id=\"F4\"><label>Fig. 4.</label><caption><p>ADT2 forms structures consistent with chloroplast division rings. (A) In addition to being expressed within chloroplasts and stromules, ADT2 was also found to accumulate in places consistent with chloroplast division rings. The top panel shows ADT2 forming rings at the equatorial plate of the chloroplast. On occasion, ADT2 was found in the constriction zone of chloroplasts (two middle panels). In these cases, the chloroplasts have a distinct dumb-bell shape and the degree of indentation depends on how far the division process has proceeded. In addition, ADT2 accumulated in a spindle-like shape that tapers at chloroplast poles (bottom panel). This fusiform ADT2 accumulation was only found at one pole of the chloroplast and is distinct from a stromule pattern shown in <a class=\"xref\" href=\"#F2\">Fig. 2</a>. (B) Schematic of chloroplast division stages: from top to bottom, positioning of chloroplast division rings; slightly constricted chloroplast just prior to division; two daughter chloroplast following division. Analogous to the fluorescent images, chloroplasts are shown in red and the position of ring proteins in blue (adapted from <a class=\"xref\" href=\"#CIT0060\">Miyagishima, 2011</a>).</p></caption>graphic: <a target=\"ext-link\" class=\"graphic\" href=\"erx02404\">erx02404</a></figure><p>To initiate division, chloroplasts have to reach a certain size (<a class=\"xref\" href=\"#CIT0067\">Pyke, 1999</a>). Therefore, we argue that chloroplasts with ADT2–CFP at the equatorial plane should be the largest as they are in the process of dividing. Conversely, chloroplasts with ADT2–CFP at their pole should be the smallest as they have just recently divided. In addition, these two classes should have little variation in size as they represent distinct stages in chloroplast development. In contrast, growing chloroplasts should vary in size as they encompass all division stages. While chloroplast volume would be the most accurate way to measure chloroplast size, it is difficult to determine. Therefore, chloroplast size was measured as the length of a chloroplast across its longest axis. Chloroplasts from uninfiltrated <i class=\"italic\">N. benthamiana</i> plants were used to determine the average size of a chloroplast because they should contain chloroplasts at many different developmental stages, and thus different sizes. Chloroplast sizes measured in three uninfiltrated plants had an average length of 5.1 μm (<a class=\"xref\" href=\"#T2\">Table 2</a>). In chloroplasts with ADT2–CFP present on a pole, the average length of these chloroplasts was significantly shorter (<i class=\"italic\">P</i><0.05), at 4.2 μm (<a class=\"xref\" href=\"#T2\">Table 2</a>). Lastly, chloroplasts with ADT2–CFP localized at the equatorial plane were significantly longer, at 6.7 μm (<i class=\"italic\">P</i><0.05; <a class=\"xref\" href=\"#T2\">Table 2</a>). As predicted, the SD from the mean was larger for chloroplasts from uninfiltrated plants, consistent with a mixed population of chloroplasts, compared with chloroplasts with ADT2–CFP at a pole or at the equatorial plane (<a class=\"xref\" href=\"#T2\">Table 2</a>). These data support the hypothesis that the ADT2 localization patterns we observed are consistent with different chloroplast division stages.</p><div class=\"table-wrap\"><label>Table 2.</label><caption><p>Comparison of chloroplast lengths</p></caption><table><thead><tr><th>\n",
       "<strong class=\"bold\">Type of chloroplast</strong> </th><th>\n",
       "<strong class=\"bold\">No. of plants</strong> </th><th>\n",
       "<strong class=\"bold\">No. of chloroplasts</strong> </th><th>\n",
       "<strong class=\"bold\">Length (µm</strong>)<sup><i class=\"italic\">a</i></sup></th><th>\n",
       "<strong class=\"bold\">SD</strong> </th></tr></thead><tbody><tr><td>Uninfiltrated</td><td>3</td><td>68</td><td>5.1</td><td>1.12</td></tr><tr><td>With polar ADT2</td><td>3</td><td>75</td><td>4.2*</td><td>0.60</td></tr><tr><td>With equatorial ADT2 ring</td><td>5</td><td>35</td><td>6.7*</td><td>0.85</td></tr></tbody></table><div class=\"table-wrap-foot\"></div></div></section><hr/><section class=\"sec\"><h2 class=\"title\">\n",
       "<i class=\"italic\">A single amino acid change in ADT2 affects chloroplast morphology and FtsZ2 localization in</i> A. thaliana\n",
       "</h2><p>Mutations in genes encoding components of the chloroplast division machinery result in changes to chloroplast morphology (<a class=\"xref\" href=\"#CIT0067\">Pyke, 1999</a>). If ADT2 has a role in chloroplast division, it is reasonable to expect that an <i class=\"italic\">adt2</i> mutant will have distorted chloroplasts. Unlike for other <i class=\"italic\">ADT</i> genes, no <i class=\"italic\">A. thaliana</i> T-DNA insertion line that abolishes <i class=\"italic\">ADT2</i> mRNA production exists (<a class=\"xref\" href=\"#CIT0011\">Corea <i class=\"italic\">et al.</i>, 2012<i class=\"italic\">b</i></a>). However, a point mutation within the coding sequence of the ACT regulatory domain of <i class=\"italic\">ADT2</i> (<i class=\"italic\">adt2-1D</i>) has been documented in which conversion of a serine to an alanine prevents allosteric inhibition of the enzyme (<a class=\"xref\" href=\"#CIT0034\">Huang <i class=\"italic\">et al.</i>, 2010</a>). To determine if <i class=\"italic\">adt2-1D</i> affects chloroplast morphology, homozygous <i class=\"italic\">adt2-1D</i> plants were examined by confocal microscopy and compared with wild-type Col-0 plants of identical age (<a class=\"xref\" href=\"#F5\">Fig. 5A</a>, <a class=\"xref\" href=\"#F5\">B</a>). Chloroplasts in <i class=\"italic\">adt2-1D</i> plants differed greatly in appearance, and were highly heterogeneous in size and shape (<a class=\"xref\" href=\"#F5\">Fig. 5C</a>). This contrasted with chloroplasts in wild-type Col-0, which were ovoid in shape and relatively uniform in size. Although many <i class=\"italic\">adt2-1D</i> chloroplasts were clearly affected by the mutation, wild-type appearing chloroplasts can still be observed, suggesting a partial loss of ADT2 function.</p><figure class=\"fig\" id=\"F5\"><label>Fig. 5.</label><caption><p>Chloroplast morphology and FtsZ2–YFP localization is affected by a point mutation in <i class=\"italic\">ADT2</i>. (A) Chloroplasts in wild-type <i class=\"italic\">A. thaliana</i> Col-0. (B) Chloroplasts in <i class=\"italic\">adt2-1D A. thaliana</i> mutants. (C) Close-ups of chloroplasts observed in <i class=\"italic\">adt2-1D</i> to show the heterogeneity in shape and size. (D) Transiently expressed FtsZ–YFP in wild-type Col-0 localizes as expected to a single ring at the equatorial plane. (E) In contrast, FtsZ2–YFP localizes as long spiralling filaments within <i class=\"italic\">adt2-1D</i> chloroplasts. (D, E) Images of chlorophyll fluorescence (left) and FtsZ2–YFP (middle) are shown separately and merged (right).</p></caption>graphic: <a target=\"ext-link\" class=\"graphic\" href=\"erx02405\">erx02405</a></figure><p>FtsZ is a tubulin-like protein that is a central component of the chloroplast division apparatus (<a class=\"xref\" href=\"#CIT0085\">Vitha <i class=\"italic\">et al.</i>, 2001</a>; <a class=\"xref\" href=\"#CIT0079\">TerBush <i class=\"italic\">et al.</i>, 2013</a>). There are two FtsZ proteins (FtsZ1 and FtsZ2), which assemble to form the earliest known division ring, the Z-ring within the stroma. As the localization of FtsZ in chloroplast division mutants has been used to provide insight into the function of putative division proteins (<a class=\"xref\" href=\"#CIT0084\">Vitha <i class=\"italic\">et al.</i>, 2003</a>; <a class=\"xref\" href=\"#CIT0022\">Glynn <i class=\"italic\">et al.</i>, 2007</a>; <a class=\"xref\" href=\"#CIT0017\">Fujiwara <i class=\"italic\">et al.</i>, 2008</a>; <a class=\"xref\" href=\"#CIT0023\">Glynn <i class=\"italic\">et al.</i>, 2009</a>; <a class=\"xref\" href=\"#CIT0061\">Nakanishi <i class=\"italic\">et al.</i>, 2009<i class=\"italic\">a</i></a>), we were interested to determine if FtsZ localization was affected in <i class=\"italic\">adt2-1D</i> plants. Thus we generated an <i class=\"italic\">FtsZ2–YFP</i> fusion construct and expressed it in <i class=\"italic\">adt2-1D</i> plants. Expression of FtsZ2–YFP in wild-type Col-0 leaves led to the formation of the expected single ring (<a class=\"xref\" href=\"#F5\">Fig. 5D</a>). In contrast, FtsZ2–YFP in <i class=\"italic\">adt2-1D</i> plants was less organized and formed what appeared to be spirals or multiple rings (<a class=\"xref\" href=\"#F5\">Fig. 5E</a>). These results demonstrate that deregulation of ADT2 abolishes proper placement of FtsZ2, further supporting an involvement of ADT2 in chloroplast division.</p></section><hr/><section class=\"sec\"><h2 class=\"title\">ADT5 is found in the nucleus</h2><p>In addition to its chloroplast localization, only ADT5–CFP was also detected in nuclei at ~4–5 dpi (<a class=\"xref\" href=\"#F2\">Figs 2A</a>, <a class=\"xref\" href=\"#F6\">6A</a>). To confirm this finding, ADT5–CFP was co-infiltrated with a YFP fusion to the nuclear marker NUP1, a component of the nuclear pore complex in <i class=\"italic\">A. thaliana</i> that was previously shown to localize to the nuclear membrane (<a class=\"xref\" href=\"#CIT0053\">Lu <i class=\"italic\">et al.</i>, 2010</a>). Confocal imaging determined that NUP1–YFP localized around ADT5–CFP (<a class=\"xref\" href=\"#F6\">Fig. 6B</a>). As NUP1–YFP localizes to the nuclear membrane, this result confirms that ADT5–CFP is contained within the nucleus and localizes uniformly throughout the nucleoplasm. As these results were obtained with constructs using a CaMV <i class=\"italic\">35S</i> promoter, we repeated the experiment and expressed ADT5–CFP under control of its native promoter (<a class=\"xref\" href=\"#F6\">Fig. 6C</a>) and confirmed the nuclear localization pattern. To ensure that the observed nuclear localization is not due to a smaller diffusible cleavage product or to a particularly high level of ADT5 compared with other ADTs, we performed western blots (<a class=\"xref\" href=\"#F6\">Fig. 6D</a>; Supplementary Fig. S2). The blot containing all ADTs expressed under the control of the CaMV <i class=\"italic\">35S</i> promoter shows a low level of cleavage in all lanes. However, more cleavage product is seen for ADT1 and ADT3 compared with ADT5, while no nuclear localization is observed with either ADT1 or ADT3. The blot showing ADT5–CFP under the control of its native promoter (<a class=\"xref\" href=\"#F6\">Fig. 6D</a>) shows a shadow band of the size of CFP, and yet a very clear nuclear localization is evident (<a class=\"xref\" href=\"#F6\">Fig. 6C</a>). These results indicate that the nuclear localization pattern is a <i class=\"italic\">bona fide</i> ADT5 localization. There are additional bands of higher molecular weight (<a class=\"xref\" href=\"#F5\">Fig. 5D</a>; Supplementary Fig. S2) that might correspond to ADT dimers or even higher multimers.</p><figure class=\"fig\" id=\"F6\"><label>Fig. 6.</label><caption><p>ADT5 is found in the nucleus. ADT5–CFP proteins are unique as they are the only full-length ADT proteins that were found in the nucleus. (A) Nuclei show a close association with chloroplasts (left) or with stromules of chloroplasts (right). Both images show ADT5–CFP within nuclei. (B) Co-localization of ADT5–CFP with NUP1–YFP. To determine if ADT5–CFP localizes to the nucleus, it was co-expressed with NUP1–YFP in <i class=\"italic\">N. benthamiana</i>. Images of chlorophyll fluorescence and ADT5–CFP are shown merged (left). NUP1–YFP is shown alone (middle) and merged with ADT5–CFP and chlorophyll fluorescence (right). NUP1–YFP localizes to the nuclear membrane and surrounds ADT5–CFP, confirming that it localizes to the nucleus. (C) ADT5–CFP transiently expressed with its native <i class=\"italic\">ADT5</i> promoter also localizes to the nucleus. (D) Western blot of ADT5–CFP (calculated size 73.9 kDa) expressed with its native promoter and visualized with a GFP antibody is detected at its expected size. As negative controls, proteins isolated from leaves transformed with GFP (25 kDa) and p19 are shown. Total soluble protein was isolated from transiently transformed leaves, and 10 μg of total soluble protein was size separated by 10% SDS–PAGE. Sizes of the protein ladder are given in kDa.</p></caption>graphic: <a target=\"ext-link\" class=\"graphic\" href=\"erx02406\">erx02406</a></figure><p>Furthermore, we often observed ADT5–CFP-containing nuclei surrounded by chloroplasts that appeared to be connected to the nucleus through stromules (<a class=\"xref\" href=\"#F6\">Fig. 6A</a>), suggesting that ADT5 nuclear localization depends on stromule-mediated transport. To test if stromule formation affects the nuclear localization of ADT5, we transiently expressed myosin XI tail domains (<i class=\"italic\">dnMyoXI-2</i> and <i class=\"italic\">dnMyoXI-K/GTD</i>) as they were previously shown to inhibit stromule formation through a dominant negative effect on wild-type myosin XI (<a class=\"xref\" href=\"#CIT0002\">Avisar <i class=\"italic\">et al.</i>, 2008</a>; <a class=\"xref\" href=\"#CIT0064\">Natesan <i class=\"italic\">et al.</i>, 2009</a>). To ensure that these treatments inhibit stromules, several control transformations were performed. To visualize stromules in control transformations, TP-ADT2–CFP was used, as it easily visualizes stromules and therefore provides a very sensitive marker to detect stromule inhibition. TP-ADT2–CFP was co-infiltrated into <i class=\"italic\">N. benthamiana</i> leaves with an empty pCB vector, a <i class=\"italic\">dnMyoXI-2</i> construct, or a <i class=\"italic\">dnMyoXI-K/GTD</i> construct (<a class=\"xref\" href=\"#CIT0002\">Avisar <i class=\"italic\">et al.</i>, 2008</a>), and the number of chloroplasts having stromules (<a class=\"xref\" href=\"#F7\">Fig. 7A</a>) and the length of stromules formed (<a class=\"xref\" href=\"#F7\">Fig. 7B</a>) were determined. In plants transformed with an empty vector, 28.7% of chloroplasts had stromules (<a class=\"xref\" href=\"#F7\">Fig. 7A</a>) and the average length of these was 4.6 μm (<a class=\"xref\" href=\"#F7\">Fig. 7B</a>). Infiltration with dnMyoXI-2 decreased the percentage of chloroplasts with stromules (22.2%; <a class=\"xref\" href=\"#F7\">Fig. 7A</a>) compared with that of the control, and significantly reduced (<i class=\"italic\">P</i><0.001) the average length of stromules to 2.7 μm (<a class=\"xref\" href=\"#F7\">Fig. 7B</a>). Treatment with dnMyoXI-K/GTD also caused a significant decrease (<i class=\"italic\">P</i><0.001) in the percentage of chloroplasts with stromules (17.3%; <a class=\"xref\" href=\"#F7\">Fig. 7A</a>) and significantly decreased (<i class=\"italic\">P</i><0.001) the average length of stromules to 3.5 μm (<a class=\"xref\" href=\"#F7\">Fig. 7B</a>). These results confirm that both myosin domains affect stromule formation.</p><figure class=\"fig\" id=\"F7\"><label>Fig. 7.\n",
       "</label><caption><p>The presence of ADT5 in the nucleus is affected by the ability to form stromules. To determine if nuclear localization of ADT5 is dependent on stromules, plants were co-infiltrated with TP-ADT2–CFP (A and B) as a control or ADT5–CFP (C) and an empty vector (dark gray), dominant negative myosin XI-2 (dnMyoXI-2; light gray) and myosin XI-K (dnMyoXI-K/GTD; white), respectively. (A) Percentage of chloroplasts having stromules. Chloroplasts were analyzed if they contained any visible TP-ADT2–CFP fluorescence and were determined to have a stromule if the projection was longer than 1 μm. In total 554, 395, and 579 chloroplasts were analyzed from plants transformed with an empty vector, dnMyoXI-2, and dnMyoXI-K/GTD, respectively. (B) Average length of stromules. A total of 166, 93, and 91 stromules were measured from plants transformed with an empty vector, dnMyoXI-2, and dnMyoXI-K/GTD, respectively. (C) Nuclear localization of ADT5–CFP. Cells were analyzed for CFP fluorescence in the nucleus only if any ADT5–CFP fluorescence was detectable. A total of 131, 190, and 358 cells were analyzed from plants transformed with an empty vector, dnMyoXI-2, and dnMyoXI-K/GTD, respectively. Each experiment was performed on three independent occasions. Significant differences (<i class=\"italic\">P</i><0.001) as determined by a one-way ANOVA (multiple comparisons) are indicated by different letters. Averages ± SE of the mean are plotted.</p></caption>graphic: <a target=\"ext-link\" class=\"graphic\" href=\"erx02407\">erx02407</a></figure><p>To determine if the ability to form stromules affects nuclear localization of ADT5, <i class=\"italic\">ADT5–CFP</i> was co-expressed with the empty pCB vector, <i class=\"italic\">dnMyoXI-2</i>, or <i class=\"italic\">dnMyoXI-K/GTD</i> (<a class=\"xref\" href=\"#F7\">Fig. 7C</a>). The extent of ADT5–CFP nuclear localization was expressed as a percentage of cells containing CFP fluorescence. Co-infiltration with the empty vector control showed that 25.9% of cells had ADT5–CFP fluorescence visible in the nucleus (<a class=\"xref\" href=\"#F7\">Fig. 7C</a>). In contrast, co-infiltration with <i class=\"italic\">dnMyoXI-2</i> or <i class=\"italic\">dnMyoXI-K/GTD</i> showed that ADT5–CFP was detected in the nucleus in only 7.1% and 4.3% of cells, respectively (<a class=\"xref\" href=\"#F7\">Fig. 7C</a>), both significant reductions (<i class=\"italic\">P</i><0.001) from the control. These data demonstrate that ADT5–CFP nuclear localization is decreased by the same conditions shown to decrease stromule formation.</p></section><hr/><section class=\"sec\"><h2 class=\"title\">\n",
       "<i class=\"italic\">ADTs in</i> A. thaliana\n",
       "</h2><p>Transient transformations using agroinfiltration are widely used in <i class=\"italic\">N. benthamiana</i> but have traditionally been difficult in <i class=\"italic\">A. thaliana</i> (<a class=\"xref\" href=\"#CIT0091\">Wroblewski <i class=\"italic\">et al.</i>, 2005</a>). We were able to transform <i class=\"italic\">A. thaliana</i> reliably by growing plants with 20 mM <span class=\"sc\">l</span>-ascorbic acid, which seemed to decrease necrosis of leaves associated with agroinfiltration. As it was not the focus of this study, the reason for this was not addressed. However, as <span class=\"sc\">l</span>-ascorbic acid can scavenge damaging reactive oxygen species (<a class=\"xref\" href=\"#CIT0019\">Gallie, 2013</a>), the increased tolerance of <i class=\"italic\">A. thaliana</i> to agroinfiltration may be due to a decrease in oxidative stress.</p><p>\n",
       "<i class=\"italic\">ADT–CFP</i> fusion genes were transiently expressed in <i class=\"italic\">A. thaliana</i> to confirm that localization in a heterologous host reflected the situation in the native environment (<a class=\"xref\" href=\"#F8\">Fig. 8</a>). As in <i class=\"italic\">N. benthamiana</i>, all ADT–CFPs in <i class=\"italic\">A. thaliana</i>, with the exception of ADT6–CFP, which appeared in the cytosol, exhibited stroma and stromule-like patterns (<a class=\"xref\" href=\"#F8\">Fig. 8A</a>). Similarly, the unique localization patterns of ADT2–CFP and ADT5–CFP were observed in <i class=\"italic\">A. thaliana</i>, localizing to the equatorial plane and poles of chloroplasts (<a class=\"xref\" href=\"#F8\">Fig. 8B</a>), and to nuclei (<a class=\"xref\" href=\"#F8\">Fig. 8C</a>), respectively. This finding was important as it verifies that the findings in <i class=\"italic\">N. benthamiana</i> are not artifacts.</p><figure class=\"fig\" id=\"F8\"><label>Fig. 8.</label><caption><p>ADT localization to the stroma and stromules, the chloroplast equatorial plane, and the nucleus can also be detected in <i class=\"italic\">A. thaliana</i>. To test if the ADT patterns determined in <i class=\"italic\">N. benthamiana</i> reflect expression in <i class=\"italic\">A. thaliana</i>, all six ADT–CFP fusion proteins were transiently expressed in <i class=\"italic\">A. thaliana</i> Col-0. All images show a merge of chlorophyll and CFP fluorescence. (A) ADT1–CFP through ADT5–CFP localize to stroma and structures resembling stromules of varying shapes and lengths, with varying levels of fluorescence in the stroma. ADT6–CFP localizes outside of chloroplasts in the cytosol. (B) Chloroplast division patterns for ADT2–CFP. (C) Nuclear localization of ADT5–CFP.</p></caption>graphic: <a target=\"ext-link\" class=\"graphic\" href=\"erx02408\">erx02408</a></figure></section></section><hr/><section class=\"sec\"><h2 class=\"title\">Discussion</h2><hr/><section class=\"sec\"><h2 class=\"title\">Phenylalanine biosynthesis and stromules</h2><p>ADTs were seen to localize to thread-like structures seemingly outside of the region of chlorophyll autofluorescence (<a class=\"xref\" href=\"#F2\">Fig. 2A</a>). Co-localization of ADT–CFP fluorescence with the stromule marker TP-ssRuBisCO–YFP (<a class=\"xref\" href=\"#CIT0065\">Nelson <i class=\"italic\">et al.</i>, 2007</a>) confirmed that all ADTs except ADT6 localized to stromules (<a class=\"xref\" href=\"#F2\">Fig. 2B</a>). Prior to our study, <i class=\"italic\">A. thaliana</i> ADTs were found uniformly throughout the stroma of chloroplasts, with no indication of stromule localization (<a class=\"xref\" href=\"#CIT0071\">Rippert <i class=\"italic\">et al.</i>, 2009</a>). However, the said study used protoplasts, which represent dedifferentiated cells that are also in a state of stress (<a class=\"xref\" href=\"#CIT0021\">Genschik <i class=\"italic\">et al.</i>, 1992</a>; <a class=\"xref\" href=\"#CIT0070\">Reyes <i class=\"italic\">et al.</i>, 2010</a>).</p><p>Stromules are dynamic structures that range in size and shape from short beak-like projections to long and elaborate tubules (<a class=\"xref\" href=\"#CIT0046\">Köhler and Hanson, 2000</a>; <a class=\"xref\" href=\"#CIT0026\">Gunning, 2005</a>; <a class=\"xref\" href=\"#CIT0024\">Gray, 2013</a>). The function of stromules has been the subject of debate, but the idea that they increase transport of compounds synthesized within plastids to other areas of the cell is now generally accepted (<a class=\"xref\" href=\"#CIT0028\">Hanson and Sattarzadeh, 2013</a>). The presence of biosynthetic enzymes, such as ADTs, in stromules is consistent with this hypothesis. Phenylalanine, the product of ADT enzymatic activity, is required within the cytosol for the synthesis of proteins and as a precursor for phenylpropanoids such as lignins and flavonoids (<a class=\"xref\" href=\"#CIT0016\">Fraser and Chapple, 2011</a>). It stands to reason that stromules could have a high concentration of phenylalanine as a result of ADT activity, providing an effective means of increasing phenylalanine export into the cytosol. Interestingly, abiotic stressors, such as drought and salt stress, known to induce stromules (<a class=\"xref\" href=\"#CIT0025\">Gray <i class=\"italic\">et al.</i>, 2012</a>), are also associated with increased flavonoid levels in leaves (<a class=\"xref\" href=\"#CIT0001\">Agati <i class=\"italic\">et al.</i>, 2011</a>; <a class=\"xref\" href=\"#CIT0059\">Mewis <i class=\"italic\">et al.</i>, 2012</a>), and genes encoding flavonoid biosynthetic enzymes are up-regulated in response to salinity-induced stress (<a class=\"xref\" href=\"#CIT0089\">Walia <i class=\"italic\">et al.</i>, 2005</a>). For example, PHENYLALANINE AMMONIA LYASE1 (PAL1) uses phenylalanine as a substrate to catalyze the first step of phenylpropanoid biosynthesis in the cytosol (<a class=\"xref\" href=\"#CIT0089\">Walia <i class=\"italic\">et al.</i>, 2005</a>; <a class=\"xref\" href=\"#CIT0016\">Fraser and Chapple, 2011</a>). Under conditions of salt stress, <i class=\"italic\">PAL1</i> up-regulation coincides with the formation of stromules, suggesting that these two events might be linked and that both events contribute to an increased transport of phenylalanine to the cytosol.</p><p>ADTs are not the only enzymes that have been associated with stromules. Other examples include geranylgeranyl diphosphate synthase (GGPS), which synthesizes geranylgeranyl diphosphate in the stromules of chloroplasts, and the compound is required in the cytosol as part of isoprenoid metabolism (<a class=\"xref\" href=\"#CIT0080\">Thabet <i class=\"italic\">et al.</i>, 2012</a>), or RuBisCO and an aspartate aminotransferase (ASP5), which are present in stromules and capable of moving between plastids via stromules when expressed as GFP fusion proteins (<a class=\"xref\" href=\"#CIT0050\">Kwok and Hanson, 2004</a>). This allows for the speculation that chloroplastic enzymes that synthesize molecules required in the cytosol preferentially localize to stromules as these might facilitate metabolite export.</p><p>\n",
       "<i class=\"italic\">In silico</i> analysis of <i class=\"italic\">A. thaliana</i> ADT sequences using ChloroP (<a class=\"xref\" href=\"#CIT0015\">Emanuelsson <i class=\"italic\">et al.</i>, 1999</a>) predicted that their N-terminal sequences were likely to encode TPs directing the enzymes to the chloroplast. We present data that confirm the <i class=\"italic\">in silico</i> prediction for ADT2, as the N-terminal portion was necessary and sufficient to allow direct import into chloroplasts and specifically stromules (<a class=\"xref\" href=\"#F3\">Fig. 3A</a>, <a class=\"xref\" href=\"#F3\">B</a>). These data also corroborate observations made for all three petunia ADTs, where the N-terminal portion of ADTs directed GFP fusion proteins to chloroplasts (<a class=\"xref\" href=\"#CIT0056\">Maeda <i class=\"italic\">et al.</i>, 2010</a>).</p></section><hr/><section class=\"sec\"><h2 class=\"title\">ADT2 and a role in chloroplast division</h2><p>Aside from its inclusion in stroma and stromules, ADT2 localized as a ring around the equatorial plane or at the poles of chloroplasts (<a class=\"xref\" href=\"#F4\">Figs 4A</a>, <a class=\"xref\" href=\"#F8\">8B</a>). The similarity of these patterns to those of chloroplast division proteins (<a class=\"xref\" href=\"#CIT0060\">Miyagishima, 2011</a>) during and after division led to an investigation of a possible second, non-enzymatic role for ADT2 in chloroplast division. Since chloroplast division is regulated by size (<a class=\"xref\" href=\"#CIT0067\">Pyke, 1999</a>), we reasoned that chloroplasts with either the equatorial or polar localization patterns would be larger and smaller, respectively, than average sized chloroplasts. This was confirmed upon comparing chloroplast lengths across their longest axis (<a class=\"xref\" href=\"#T2\">Table 2</a>). Additionally, the SD of chloroplast lengths varied. It was lowest in chloroplasts with ADT2 at the equator or at a pole, in agreement with these chloroplasts being in very distinct phases, either just prior to or post-division, and therefore very similar in size. This was in contrast to chloroplasts from uninfiltrated plants, which are comprised of chloroplasts in all division states. Therefore, our results suggest that ADT2 localizes to the division plane early in the division process and remains there throughout the duration of constriction and separation into daughter organelles. Similarly, known division proteins such as FtsZ and ACCUMULATION AND REPLICATION OF CHLOROPLASTS 6 (ARC6) assemble at the equatorial plane in the process leading to constriction and division (<a class=\"xref\" href=\"#CIT0085\">Vitha <i class=\"italic\">et al.</i>, 2001</a>; <a class=\"xref\" href=\"#CIT0084\">2003</a>).</p><p>The striking similarities between ADT2 and other chloroplast division proteins prompted observation of chloroplasts in <i class=\"italic\">adt2</i> mutant plants. Interestingly, no T-DNA insertion knockout lines that abolish <i class=\"italic\">ADT2</i> mRNA are available (<a class=\"xref\" href=\"#CIT0011\">Corea <i class=\"italic\">et al.</i>, 2012<i class=\"italic\">b</i></a>). This makes ADT2 unique and raises the possibility that an <i class=\"italic\">adt2</i> knockout is lethal. However, plants homozygous for the <i class=\"italic\">adt2-1D</i> point mutation have been documented (<a class=\"xref\" href=\"#CIT0034\">Huang <i class=\"italic\">et al.</i>, 2010</a>). The appearance of <i class=\"italic\">adt2-1D</i> chloroplasts was variable and the presence of misshapen and heterogeneous chloroplasts is consistent with previous descriptions of chloroplast morphology in division mutants (<a class=\"xref\" href=\"#CIT0068\">Pyke and Leech, 1992</a>; <a class=\"xref\" href=\"#CIT0007\">Colletti <i class=\"italic\">et al.</i>, 2000</a>; <a class=\"xref\" href=\"#CIT0023\">Glynn <i class=\"italic\">et al.</i>, 2009</a>; <a class=\"xref\" href=\"#CIT0062\">Nakanishi <i class=\"italic\">et al.</i>, 2009<i class=\"italic\">b</i></a>). In the <i class=\"italic\">adt2-1D</i><a class=\"xref\" href=\"#CIT0079\">TerBush <i class=\"italic\">et al.</i>, 2013</a>). Given the central role that FtsZ proteins play in division, we reasoned that FtsZ localization should be affected in <i class=\"italic\">adt2-1D</i> chloroplasts if ADT2 is a chloroplast division protein. Expression of an <i class=\"italic\">FtsZ2–YFP</i> fusion construct in <i class=\"italic\">adt2-1D</i> plants revealed long and spiraling FtsZ2–YFP filaments throughout the chloroplast stroma. Although FtsZ2–YFP was overexpressed in our study, in wild-type Col-0 chloroplasts the fusion protein localized as expected as a single equatorial ring. We propose that the abnormal appearance of FtsZ2–YFP within <i class=\"italic\">adt2-1D</i> chloroplasts suggests that ADT2 regulates FtsZ positioning. However, we cannot ignore the possibility that elevated phenylalanine levels in <i class=\"italic\">adt2-1D</i> (up to 160-fold compared with the wild type; <a class=\"xref\" href=\"#CIT0034\">Huang <i class=\"italic\">et al.</i>, 2010</a>) are at least in part responsible for the observed changes.</p><p>It is intriguing to note that not all ring proteins have been identified. Although FtsZ, one of the inner rings, and ARC5 (dynamin), one of the outer rings, have been known for some time (<a class=\"xref\" href=\"#CIT0085\">Vitha <i class=\"italic\">et al.</i>, 2001</a>; <a class=\"xref\" href=\"#CIT0020\">Gao <i class=\"italic\">et al.</i>, 2003</a>), the identity of the outer plastid-dividing (PD) ring (<a class=\"xref\" href=\"#CIT0095\">Yoshida <i class=\"italic\">et al.</i>, 2010</a>), polyglucan filaments, has just recently been suggested, and the composition of the inner PD ring is still unknown. The realization that enzymes can have a second unrelated, non-enzymatic function, even as part of cellular structural components, is discovered more and more frequently (<a class=\"xref\" href=\"#CIT0035\">Huberts and van der Klei, 2010</a>; MoonProt Database: <a target=\"uri\" class=\"uri\" href=\"www.moonlightingproteins.org\"> www.moonlightingproteins.org, last accessed 13 February 2017</a>). A good example of this is the <i class=\"italic\">Physcomitrella patens</i> enzyme presenilin, the catalytic unit for γ-secretase, which has an independent function in the cytoskeletal network (<a class=\"xref\" href=\"#CIT0043\">Khandelwal <i class=\"italic\">et al.</i>, 2007</a>). However, additional studies will be required to determine the precise relationship between ADT2, phenylalanine levels, chloroplast division, and other components of the chloroplast division machinery.</p></section><hr/><section class=\"sec\"><h2 class=\"title\">ADT5 and the nucleus</h2><p>Similar to ADT2, ADT5 has an additional unique localization pattern and was clearly observed in the nuclei of both <i class=\"italic\">N. benthamiana</i> and <i class=\"italic\">A. thaliana</i> (<a class=\"xref\" href=\"#F6\">Fig. 6A</a>, <a class=\"xref\" href=\"#F6\">C</a>). Proteins with dual plastid and nuclear localization may be significant in the context of retrograde signaling. While retrograde signaling traditionally refers to chemical messengers that are released from plastids and affect nuclear gene expression (<a class=\"xref\" href=\"#CIT0036\">Inaba <i class=\"italic\">et al.</i>, 2011</a>), it is becoming apparent that proteins within the chloroplast can also act as retrograde signals (<a class=\"xref\" href=\"#CIT0037\">Isemer <i class=\"italic\">et al.</i>, 2012</a>; <a class=\"xref\" href=\"#CIT0049\">Krause <i class=\"italic\">et al.</i>, 2012</a>). One such protein is WHIRLY1 from <i class=\"italic\">A. thaliana</i>, which can move directly from plastids to the nucleus (<a class=\"xref\" href=\"#CIT0037\">Isemer <i class=\"italic\">et al.</i>, 2012</a>). In plastids, WHIRLY1 contributes to plastid genome stability by preventing illegitimate recombination (<a class=\"xref\" href=\"#CIT0058\">Maréchal <i class=\"italic\">et al.</i>, 2009</a>). In the nucleus, it acts as a transcriptional activator of pathogen response genes (<a class=\"xref\" href=\"#CIT0037\">Isemer <i class=\"italic\">et al.</i>, 2012</a>), consistent with the increased pathogen susceptibility associated with decreased WHIRLY1 DNA binding ability (<a class=\"xref\" href=\"#CIT0012\">Desveaux <i class=\"italic\">et al.</i>, 2005</a>). Whether ADT5 has a role in retrograde signaling is currently unknown. In addition, many other enzymes with diverse functions have been reported in both plastids and nuclei, such as phosphate-isopentyltransferase 3, an enzyme involved in cytokinin biosynthesis (<a class=\"xref\" href=\"#CIT0018\">Galichet <i class=\"italic\">et al.</i>, 2008</a>; <a class=\"xref\" href=\"#CIT0049\">Krause <i class=\"italic\">et al.</i>, 2012</a>), CDT1, a kinase required in cell cycle regulation (<a class=\"xref\" href=\"#CIT0069\">Raynaud <i class=\"italic\">et al.</i>, 2005</a>), and a dihydrofolate reductase required for nucleotide metabolism (<a class=\"xref\" href=\"#CIT0054\">Luo <i class=\"italic\">et al.</i>, 1997</a>), but often the nuclear role of these enzymes is ill defined.</p><p>While a direct mechanism of protein transport between plastids and the nucleus through stromules is a hypothetical mode (<a class=\"xref\" href=\"#CIT0049\">Krause <i class=\"italic\">et al.</i>, 2012</a>), they have been shown to interconnect plastids (<a class=\"xref\" href=\"#CIT0045\">Köhler <i class=\"italic\">et al.</i>, 1997</a>; <a class=\"xref\" href=\"#CIT0050\">Kwok and Hanson, 2004</a>; <a class=\"xref\" href=\"#CIT0028\">Hanson and Sattarzadeh, 2013</a>). Chloroplasts with stromules containing ADT5–CFP often appeared to connect directly with the nucleus (<a class=\"xref\" href=\"#F6\">Fig. 6A</a>). Expression of dominant negative forms of myosin XI were found to inhibit stromules (<a class=\"xref\" href=\"#F7\">Fig. 7A</a>, <a class=\"xref\" href=\"#F7\">B</a>) and also significantly reduce ADT5–CFP localization to the nucleus (<a class=\"xref\" href=\"#F7\">Fig. 7C</a>), providing indirect evidence of stromule-mediated nuclear transport. We are aware that there are alternative interpretations of these results, as myosin XI is involved in other processes including movement of organelles (<a class=\"xref\" href=\"#CIT0002\">Avisar <i class=\"italic\">et al.</i>, 2008</a>) and cytoplasmic streaming (<a class=\"xref\" href=\"#CIT0075\">Shimmen and Yokota, 2004</a>). Regardless of this, the appearance of stromules directly connecting to nuclei (<a class=\"xref\" href=\"#F6\">Fig. 6A</a>) makes the possibility of a stromule-mediated nuclear transport system intriguing.</p><p>Currently, the role that ADT5 plays in the nucleus is unknown. It is conceivable that it acts as a transcriptional regulator of <i class=\"italic\">ADT</i> genes or other genes within the same biosynthetic pathway. There is precedence for an enzyme to act as a transcriptional regulator of functionally related genes. For example, <i class=\"italic\">A. thaliana</i> HEXOKINASE1 (HXK1) is involved in glucose metabolism in mitochondria, but also localizes to the nucleus where it forms part of a protein complex affecting transcription of genes involved in glucose signaling (<a class=\"xref\" href=\"#CIT0006\">Cho <i class=\"italic\">et al.</i>, 2006</a>). Furthermore, in the budding yeast <i class=\"italic\">Saccharomyces cerevisiae</i>, ARG5,6, an enzyme involved in arginine biosynthesis, is able to bind DNA directly and regulate gene expression (<a class=\"xref\" href=\"#CIT0027\">Hall <i class=\"italic\">et al.</i>, 2004</a>). Interestingly, mutant analysis has shown that loss of ADT5 activity cannot be compensated for by other ADTs and is the only single <i class=\"italic\">ADT</i> knockout with a visible phenotype (<a class=\"xref\" href=\"#CIT0011\">Corea <i class=\"italic\">et al.</i>, 2012<i class=\"italic\">b</i></a>), consistent with a unique nuclear role for ADT5.</p></section><hr/><section class=\"sec\"><h2 class=\"title\">ADT2 and ADT5: moonlighting proteins</h2><p>The term ‘moonlighting protein’ was coined to describe proteins that perform multiple autonomous and often unrelated functions without these functions being partitioned into different domains of the protein or resulting from alternative splicing or gene fusion (reviewed in <a class=\"xref\" href=\"#CIT0039\">Jeffery, 1999</a>, <a class=\"xref\" href=\"#CIT0040\">2013</a>). Since the description of the first moonlighting proteins, the number of documented moonlighting proteins has increased to ~300 (<a class=\"xref\" href=\"#CIT0040\">Jeffery, 2013</a>; MoonProt Database: <a target=\"uri\" class=\"uri\" href=\"www.moonlightingproteins.org\"> www.moonlightingproteins.org</a>). Enzymes are common among moonlighting proteins and many have additional non-enzymatic functions, including roles as structural components and as regulators of transcription or translation (<a class=\"xref\" href=\"#CIT0040\">Jeffery, 2013</a>). For example, in <i class=\"italic\">Tetrahymena</i>, a citrate synthase acts as an enzyme in mitochondria while in the cytosol it can polymerize to form 14 nm filaments and then act as a cytoskeletal protein (<a class=\"xref\" href=\"#CIT0047\">Kojima <i class=\"italic\">et al.</i>, 1997</a>). We propose that ADT2 and ADT5 are moonlighting proteins. In the current study we provide evidence that ADT2, with demonstrated arogenate dehydratase activity (<a class=\"xref\" href=\"#CIT0005\">Cho <i class=\"italic\">et al.</i>, 2007</a>), can form rings around chloroplasts similar to FtsZ or ARC5 as part of their role in chloroplast division (<a class=\"xref\" href=\"#CIT0085\">Vitha <i class=\"italic\">et al.</i>, 2001</a>; <a class=\"xref\" href=\"#CIT0020\">Gao <i class=\"italic\">et al.</i>, 2003</a>). The dual localization of ADT5 to chloroplasts and nuclei suggests that ADT5 has an additional role in the nucleus, possibly in transcriptional regulation. As the entire ADT enzyme family catalyzes the final step in phenylalanine biosynthesis, we expect this to be a key regulatory step.</p><p>Moonlighting proteins appear to be ubiquitous in nature, with documented examples in simple single-cell organisms, such as archaea and bacteria, and complex eukaryotes including plants and animals. It seems likely that moonlighting proteins are evolutionarily advantageous, ensuring that the number of genes in a genome does not limit the number of functions they are capable of performing. Many moonlighting proteins are ancient in terms of their evolutionary history, giving them ample time to adapt to a second role (<a class=\"xref\" href=\"#CIT0040\">Jeffery, 2013</a>). Analyzing the subcellular localization of <i class=\"italic\">A. thaliana</i> ADTs shows that the function of enzymes is far more complex than previously realized.</p></section></section><hr/><section class=\"sec-supplementary-material\"><h2 class=\"title\">Supplementary data</h2><p>Supplementary data are available at <i class=\"italic\">JXB</i> online.</p><p>Fig. S1. Negative controls.</p><p>Fig. S2. Western blots showing expression of transiently expressed ADTs.</p></section><hr/><section class=\"sec-supplementary-material\"><h2 class=\"title\">Supplementary Material</h2></section></main></article>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x =  EPMC.from_pmcid(p.pmcid)\n",
    "html = x.html()\n",
    "if x.missing:\n",
    "    print(x.missing)\n",
    "HTML(CSS + html)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f333636d-675c-44d3-a609-d666f27ace47",
   "metadata": {},
   "outputs": [],
   "source": [
    "# x.save_content('saved.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "826f8bef-5533-41cc-b6ad-e419bd16f3b0",
   "metadata": {},
   "outputs": [],
   "source": [
    "x.save_content(\"saved.xml\", pretty=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "1b3b3e9c-579c-4601-b7aa-74d8c97e51fb",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "## Highlight\n",
       "\n",
       "\n",
       "We show subcellular localization of all six *Arabidopsis thaliana* arogenate dehydratases to chloroplasts and stromules. Furthermore, ADT2 and ADT5 have additional localization patterns suggesting non-enzymatic functions and roles as moonlighting proteins.\n",
       "\n",
       "\n",
       "**Keywords:** Arogenate dehydratase, chloroplast division, moonlighting proteins, nuclear localization, phenylalanine biosynthesis, stromules.\n",
       "\n",
       "## Abstract\n",
       "\n",
       "\n",
       "Arogenate dehydratases (ADTs) catalyze the final step in phenylalanine biosynthesis in plants. The *Arabidopsis thaliana* genome encodes a family of six ADTs capable of decarboxylating/dehydrating arogenate into phenylalanine. Using cyan fluorescent protein (CFP)-tagged proteins, the subcellular localization patterns of all six *A. thaliana* ADTs were investigated in intact *Nicotiana benthamiana* and *A. thaliana* leaf cells. We show that *A. thaliana* ADTs localize to stroma and stromules (stroma-filled tubules) of chloroplasts. This localization pattern is consistent with the enzymatic function of ADTs as many enzymes required for amino acid biosynthesis are primarily localized to chloroplasts, and stromules are thought to increase metabolite transport from chloroplasts to other cellular compartments. Furthermore, we provide evidence that ADTs have additional, non-enzymatic roles. ADT2 localizes in a ring around the equatorial plane of chloroplasts or to a chloroplast pole, which suggests that ADT2 is a component of the chloroplast division machinery. In addition to chloroplasts, ADT5 was also found in nuclei, again suggesting a non-enzymatic role for ADT5. We also show evidence that ADT5 is transported to the nucleus via stromules. We propose that ADT2 and ADT5 are moonlighting proteins that play an enzymatic role in phenylalanine biosynthesis and a second role in chloroplast division or transcriptional regulation, respectively.\n",
       "\n",
       "## Introduction\n",
       "\n",
       "\n",
       "Arogenate dehydratases (ADTs; EC 4.2.1.911) catalyze the final step of phenylalanine biosynthesis through decarboxylation/dehydration of arogenate to form the aromatic amino acid phenylalanine ([Fig. 1A](#F1); [Laskar *et al.*, 2010](#CIT0051); [Tzin and Galili, 2010](#CIT0082)). In plants phenylalanine serves as a precursor not only of proteins but also of many secondary metabolites, including phenylpropanoids ([Herrmann and Weaver, 1999](#CIT0032); [Knaggs, 2003](#CIT0044); [Vogt, 2010](#CIT0087)). Phenylpropanoids have diverse functions, including structural support, pigmentation, and scent formation ([Vogt, 2010](#CIT0087)), indicating the great importance of phenylalanine biosynthesis in plants.\n",
       "\n",
       "\n",
       "### Fig. 1.\n",
       "\n",
       "\n",
       "[![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bb7/5444438/e505d73fccce/erx02401.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5444438_erx02401.jpg)\n",
       "\n",
       "\n",
       "\n",
       "[Open in a new tab](figure/F1/)\n",
       "\n",
       "\n",
       "\n",
       "Phenylalanine synthesis, arogenate dehydratases, and stromules. (A) Phenylalanine can be synthesized in plants using either the prephenate (top) or the arogenate (bottom) pathway ([Cho *et al.*, 2007](#CIT0005); [Maeda and Dudareva, 2012](#CIT0055)). Prephenate is either decarboxylated/dehydrated to phenylpyruvate (PP) by a prephenate dehydratase (PDT) and PP is then transaminated by a phenylpyruvate aminotransferase (PPAT) to phenylalanine. Alternatively the two enzymatic steps are reversed, whereby prephenate is transaminated to arogenate by a prephenate aminotransferase (PAT) and arogenate is then decarboxylated/dehydrated to phenylalanine by an arogenate dehydratase (ADT). (B) *A. thaliana ADT* constructs were cloned in different lengths. The full-length (F) sequence represents the entire *ADT* ORF while an N-terminal construct only includes the transit peptide (TP). (C) Schematic diagram of a chloroplast showing the formation of stromules. Stromules are stroma-filled protrusions of the outer and inner membrane from chloroplasts. They can differ in length, forming long thread-like extensions or globular structures.\n",
       "\n",
       "\n",
       "\n",
       "The *Arabidopsis thaliana* genome encodes a small gene family of six *ADT* genes, all sharing similar sequences and domain structures ([Ehlting *et al.*, 2005](#CIT0014); [Cho *et al.*, 2007](#CIT0005)). Families of ADTs are common in plant genomes and have been identified in both monocot and dicot species ([Tuskan *et al.*, 2006](#CIT0081); [Yamada *et al.*, 2008](#CIT0094); [Maeda *et al.*, 2010](#CIT0056), [2011](#CIT0057)). The presence of multiple isoforms suggests that ADTs might have evolved different properties, and/or that each ADT is either transcriptionally or post-translationally regulated to allow for distinct functional roles. For example, all six *A. thaliana* ADTs accept arogenate as a substrate, as their name suggests, but two of the six ADTs also accept prephenate [meaning they can act as ADTs and prephenate dehydratases (PDTs; [Fig. 1A](#F1); [Cho *et al.*, 2007](#CIT0005); [Bross *et al.*, 2011](#CIT0004))]. Furthermore, ADTs differentially contribute to lignin content and bolting/flowering transition. Analysis of different *A. thaliana* ADT knockout mutants has indicated that specific ADTs preferentially contribute to the synthesis of different downstream products of the phenylpropanoid pathway ([Corea *et al.*, 2012*b*](#CIT0011)). In lines harboring an adt5 knockout, the amount of phenylalanine, and its proportions relative to tyrosine and tryptophan, were lower in stem tissues compared with the wild-type control ([Corea *et al.*, 2012*a*](#CIT0010)). Together, these data suggest that ADT5 plays a predominant role in phenylalanine biosynthesis for lignin deposition in stems. Also, the *adt1*, but not the *adt4*, knockout line exhibited a resistant late bolting/flowering phenotype compared with wild-type *A. thaliana* under different environmental conditions, which is consistent with a decreased level of *ADT* expression after cold treatment in resistant late bolting/flowering *Beta vulgaris altissima* ([Hébrard *et al.*, 2013](#CIT0030)). This suggests that ADTs may be functional targets of DNA methylation in the shoot apical meristem during vernalization, and that the accumulation of phenolic compounds may play a role in floral transition.\n",
       "\n",
       "\n",
       "Plant ADTs, including all six *A. thaliana* isoforms, have three domains ([Fig. 1B](#F1)), a putative N-terminal transit peptide (TP), an internal catalytic domain, and a C-terminal ACT (aspartokinase–chorismate mutase–TyrA) domain ([Cho *et al.*, 2007](#CIT0005)). Both the catalytic and ACT domains are conserved across plant, bacterial, and fungal ADTs and PDTs, with the catalytic domain decarboxylating/dehydrating prephenate and/or arogenate ([Cho *et al.*, 2007](#CIT0005); [Bross *et al.*, 2011](#CIT0004)) while the ACT domain is involved in allosteric regulation induced by ligand binding ([Tan *et al.*, 2008](#CIT0078); [Vivan *et al.*, 2008](#CIT0086)). The N-terminal domain is unique to plant ADTs and is not found in the bacterial or fungal proteins. In *A. thaliana* ADTs, this domain is ~100–130 amino acids in length, and are likely to be chloroplast TPs according to sequence prediction programs, which is consistent with phenylalanine biosynthesis occurring in chloroplasts ([Jung *et al.*, 1986](#CIT0041); [Cho *et al.*, 2007](#CIT0005); [Li and Chiu, 2010](#CIT0052)) and a chloroplastic localization identified for ADTs in protoplasts ([Rippert *et al.*, 2009](#CIT0071)).\n",
       "\n",
       "\n",
       "Identifying the subcellular localization of proteins can help to define their functional role and has subsequently led to the identification of new unexpected roles ([Sparkes and Brandizzi, 2012](#CIT0077)). This approach can be particularly helpful when dissecting and differentiating the biological roles of members within protein families ([Karve *et al.*, 2008](#CIT0042)). In this study, we provide evidence that most *A. thaliana* ADTs are targeted to the stroma and stromules (stroma-filled tubules; [Köhler and Hanson, 2000](#CIT0046)) of chloroplasts and we show that this targeting is dependent on the presence of the TP. This subcellular localization is consistent with the enzymatic role of ADTs in phenylalanine biosynthesis and the proposed role of stromules in increasing metabolite transport ([Natesan *et al.*, 2005](#CIT0063)). In addition, we demonstrate that two of the ADTs, ADT2 and ADT5, have additional subcellular localization patterns that suggest novel, non-enzymatic functions.\n",
       "\n",
       "## Materials and methods\n",
       "\n",
       "\n",
       "### Growth conditions for bacteria and plants\n",
       "\n",
       "\n",
       "\n",
       "*Escherichia coli* DH5α and DH10β strains (Invitrogen catalog nos 11319019 and 18290015, respectively) were used for the maintenance and amplification of plasmid DNA. *Agrobacterium tumefaciens* strain LBA4404 containing Ti helper plasmid pAL4404 (NCCB accession no. PC2760; [Hoekema *et al.*, 1983](#CIT0033); [Hellens *et al.*, 2000](#CIT0031)) was used for transformation of *Nicotiana benthamiana* and *A. thaliana*. *Agrobacterium tumefaciens* strain GV3101 ([Koncz and Schell, 1986](#CIT0048); [Hellens *et al.*, 2000](#CIT0031)) was used for the transformation of the *dominant negative myosin XI-2* (*dnMyoXI-2*) and *dnMyoXI-K/GTD* constructs ([Avisar *et al.*, 2008](#CIT0002)). *Escherichia coli* and *A. tumefaciens* were grown at 37°C in LB medium, and at 28°C in YEB medium, respectively ([Vervliet *et al.*, 1975](#CIT0083); [Bertani, 2004](#CIT0003)), with media supplemented with appropriate antibiotics.\n",
       "\n",
       "\n",
       "Three- to five-week-old *N. benthamiana* were used for localization studies of *A. thaliana* ADTs and grown in incubators (Conviron) under 16 h light (80–100 µmol m−2 s−1) and 8 h dark, with light and dark temperatures set to 24°C and 22°C, respectively. *Arabidopsis thaliana* accession Columbia-0 (Col-0) was grown for 3–4 weeks with the same photoperiod and a light intensity of 150 µmol m−2 s−1. *Arabidopsis thaliana* plants grown for transient transformations were watered with a 20 mM l-ascorbic acid solution.\n",
       "\n",
       "\n",
       "The *adt2-1D* mutation is an ethyl methanesulfonate-induced point mutation that causes a serine to be replaced by an alanine in the ACT regulatory domain, leading to an enzyme that is unable to respond to phenylalanine-mediated allosteric inhibition ([Huang *et al.*, 2010](#CIT0034)).\n",
       "\n",
       "### Cloning of ADT–CFP fusion constructs\n",
       "\n",
       "\n",
       "Primers were designed ([Table 1](#T1)) to amplify full-length *A. thaliana ADT* genes (*ADT1*, *At1g11790*; *ADT2*, *At3g07630*; *ADT3*, *At2g27820*; *ADT4*, *At3g44720*; *ADT5*, *At5g22630*; and *ADT6*, *At1g08250*) ([Ehlting *et al.*, 2005](#CIT0014); [Cho *et al.*, 2007](#CIT0005)) and tested (Lynnon BioSoft, Version 6). Amplified full-length *ADT* sequences contain the entire ORF, including the coding sequence for TPs, ADT/PDT catalytic and ACT regulatory domains. In addition, ADT2 was cloned as the TP sequence only ([Fig. 1B](#F1)). Primers were designed to include restriction enzyme cleavage and docking sites, to allow for directional integration of PCR fragments into the target vector ([Table 1](#T1)). *ADT* sequences were amplified with Platinum *Taq* Polymerase High Fidelity (Invitrogen catalog no. 11304011) with previously cloned *ADT* sequences as templates ([Cho *et al.*, 2007](#CIT0005)).\n",
       "\n",
       "\n",
       "#### Table 1.\n",
       "\n",
       "\n",
       "\n",
       "List of primer sequences\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "| **Name** ***a*** | **Sequence (5'–3'**)***b*** | **Restriction enzyme recognition sequence** |\n",
       "| --- | --- | --- |\n",
       "| CFP-For | *AT* **CGGACCG**GTCGCCACC ATGGTGAGCAAGG | *Cpo*I |\n",
       "| CFP-Rev | *TCA* **TCTAGA**TTACTTGTA CAGCTCGTCC | *Xba*I |\n",
       "| CFP-Seq | GATCTGAGCTACACATGC | N/A |\n",
       "| ADT1-F | **AAGCTT**ATGGCTCTGAGGTGTTTTC | *Hin*dIII |\n",
       "| ADT1-R | **GGATCC**TGTCTGACTAGATCCATTGG | *Bam*HI |\n",
       "| ADT2-F | **AAGCTT**ATGGCAATGCACACTGTTCG | *Hin*dIII |\n",
       "| ADT2-S | **AAGCTT**ATGCGTGTTGCGT ATCAGGGAGTACG | *Hin*dIII |\n",
       "| ADT2-R | **GGATCC**AAGAGCATTGTA GTGTCCACTGG | *Bam*HI |\n",
       "| ADT2-RTP | **GGATCC**TTAACGCGGGAGCCATTAG | *Bam*HI |\n",
       "| ADT3-F | **GAATTC**ATGAGAACTCTCTTACCTTC | *Eco*RI |\n",
       "| ADT3-R | **GGATCC**ATCAATGAAAATGTTGATGACG | *Bam*HI |\n",
       "| ADT4-F | **CTCGAG**ATGCAAGCCGCAACGTCG | *Xho*I |\n",
       "| ADT4-R | **GGATCC**AATGCTTCTTCT GTGGATGTCATGG | *Bam*HI |\n",
       "| ADT5-F | **CTCGAG**ATGCAAACCATTTCGCC | *Xho*I |\n",
       "| ADT5-R | **CCCGGG**TTACGTCTTCGCTAG | *Sma*I |\n",
       "| ADT6-F | **GAATTC**ATGAAAGCTCTATCATC | *Eco*RI |\n",
       "| ADT6-R | **GGATCC**ATCGATGAAGTTGATG | *Bam*HI |\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "[Open in a new tab](table/T1/)\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "*a* CFP-For/Rev, amplify cerulean cyan fluorescent protein sequence; CFP-Seq, pCB sequencing primer; F, complementary to the 5' end of the full-length *ADT* coding sequence; S, complementary to the 5' end of the catalytic domain; R, reverse primer complementary to the 3' end of the *ADT* coding sequence; RTP, reverse primer complementary to the 3' end of the transit peptide.\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "*b* Italics, restriction enzyme docking sites; bold, restriction enzyme recognition sequence, underline, nucleotides to maintain frame; dotted underline, pEZT-NL vector sequence; double underline, introduced start or stop codon; unformatted, original, unmodified template sequence.\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "The T-DNA binary vector pEZT-NL (Carnegie Cell Imaging Project, [http://deepgreen.stanford.edu, last accessed 13 February 2017](http://deepgreen.stanford.edu)) was used in conjunction with the pAL4404 Ti-helper plasmid for expression *in planta*. *ADT* genes expressed from pEZT-NL are under the control of the *Cauliflower mosaic virus* (CaMV) *35S* promoter and are translated as C-terminally tagged enhanced green fluorescent protein (EGFP) fusions. For co-localization studies, pEZT-NL was modified by replacing *EGFP*, flanked by *Cpo*I and *Xba*I sites, with the cerulean cyan fluorescent protein (CFP)-coding sequence ([Conley *et al.*, 2009*b*](#CIT0009)). Both PCR-amplified *CFP* sequences and the empty pEZT-NL were double-digested with *Cpo*I and *Xba*I; the resulting fragments were ligated and then transformed into *E. coli*. Positive transformants were selected on LB medium containing gentamicin. The resulting vector was renamed pCB. Using the appropriate restriction enzymes, *ADT* genes were cloned into pCB, and all resulting pCB-ADT vectors were sequenced to ensure proper fusion and sequence integrity of the *ADT–CFP* sequences.\n",
       "\n",
       "\n",
       "To clone the native *ADT5* promoter (proNat5), 1 kb upstream of the *ADT5* start codon was PCR amplified with primers that added a 5' *Mau*BI and a 3' *Xho*I restriction site. The amplified *Mau*BI–*Xho*I fragment was used to replace the CaMV *35S* promoter in pCB-ADT5, generating the vector proNat5::ADT5:CFP.\n",
       "\n",
       "### Cloning filamentous temperature sensitive Z2 (FtsZ2)–yellow fluorescent protein (YFP)\n",
       "\n",
       "\n",
       "The Gateway-compatible vector pLIC6 encoding *FtsZ2-1* cDNA was obtained from the ABRC (stock number DKLAT2G36250; [Popescu *et al.*, 2007](#CIT0066)). Restriction digest of pLIC6 with *Hin*dIII yielded a 2329 bp restriction fragment containing *FtsZ2-1* flanked by *attachment*B (*att*B) sites. This gel-purified fragment was then recombined into pDONR221 ([Hartley *et al.*, 2000](#CIT0029)). The entry clone created was digested with *Ase*I and the expected 2608 bp fragment encoding *FtsZ2-1* flanked by *att*L sites was isolated and recombined into the destination vector pEarleygate101 ([Earley *et al.*, 2006](#CIT0013)), generating an *FtsZ2–YFP* fusion construct with expression regulated by the CaMV *35S* promoter.\n",
       "\n",
       "### Bacterial transformations\n",
       "\n",
       "\n",
       "Plasmid DNA was isolated from overnight *E. coli* cultures using an alkaline lysis method (modified from [Ish-Horowicz and Burke, 1981](#CIT0038); [Sambrook and Russell, 2001](#CIT0072)) and transformed into electrocompetent *E. coli* (ElectroMax DH5α, Invitrogen), or electrocompetent *A. tumefaciens* ([Wise *et al.*, 2006](#CIT0090)), using the Gene Pulser II System (Bio-Rad) set to 2.0 kV, 25 µF capacitance, and 200 Ω or 400 Ω resistance, respectively. Immediately following electroporation, *E. coli* and *A. tumefaciens* cells were incubated for 1 h in non-selective LB medium, before plating on selective medium. Correct insertion of amplicons into plasmid DNA of positive *E. coli* transformants was confirmed by restriction enzyme digestion and sequencing of isolated plasmid DNA. pCB-ADTs were transformed into *A. tumefaciens* LBA4404 containing Ti-helper plasmid pAL4404 ([Hoekema *et al.*, 1983](#CIT0033); [Hellens *et al.*, 2000](#CIT0031)).\n",
       "\n",
       "### Organelle markers\n",
       "\n",
       "\n",
       "To identify stromules, the TP of the small subunit of tobacco RuBisCO fused to YFP (TP-ssRuBisCO–YFP; [Nelson *et al.*, 2007](#CIT0065)) was used in co-localization experiments. This fusion construct is under the control of a CaMV *35S* promoter and, after translation, the TP guides the fusion protein to the chloroplast stroma where it can be used to identify stromules ([Nelson *et al.*, 2007](#CIT0065)).\n",
       "\n",
       "\n",
       "The T-DNA-containing binary vector pEarleygate301-YFP, encoding *A. thaliana NUCLEOPORIN1* fused to *YFP* (*NUP1–YFP*) is under the expression of its native promoter ([Lu *et al.*, 2010](#CIT0053)) and was used as a nuclear marker.\n",
       "\n",
       "\n",
       "\n",
       "*Agrobacterium tumefaciens* GV3101 containing pCB302 encoding the dominant negative form of either *N. benthamiana* myosin XI-2 or myosin XI-K/GTD was used to inhibit stromule formation ([Avisar *et al.*, 2008](#CIT0002); [Natesan *et al.*, 2009](#CIT0064)). Each construct encodes the globular tail domain of the corresponding myosin XI. Expression of the dominant negative constructs in pCB302 is regulated *in planta* by the nopaline synthase promoter ([Xiang *et al.*, 1999](#CIT0093); [Avisar *et al.*, 2008](#CIT0002)).\n",
       "\n",
       "### Agroinfiltration of tobacco leaves\n",
       "\n",
       "\n",
       "\n",
       "*Nicotiana benthamiana* and *A. thaliana* were transiently transformed by pressure-infiltrating cultures of *A. tumefaciens* ([Wroblewski *et al.*, 2005](#CIT0091); [Wydro *et al.*, 2006](#CIT0092); [Conley *et al.*, 2009*b*](#CIT0009)). *Agrobacterium tumefaciens* were grown overnight in 3 ml of YEB medium with the appropriate antibiotics. Then 50 μl of the overnight culture was transferred to 50 ml of YEB containing 25 μl of 200 mM acetosyringone and 500 μl of 1 M MES, and grown until cell density reached an OD600 of 0.5–0.8. Cells were collected by centrifugation, resuspended in Gamborg’s solution to a final OD600 of 1.0, and incubated for an additional hour prior to infiltration. For co-infiltration, equal volumes of *A. tumefaciens* cultures containing different vectors were combined to maintain a final OD600 of 1.0. The p19 vector encodes a 19 kDa protein from *Tomato bushy stunt virus*, which has been shown to enhance transgene expression through suppression of post-transcriptional gene silencing (PTGS; [Silhavy *et al.*, 2002](#CIT0076); [Voinnet *et al.*, 2003](#CIT0088)) and was added to all transient transformations.\n",
       "\n",
       "\n",
       "A minor variation of this protocol was used for co-expression with *TP-ssRuBisCO–YFP* as this construct produced a very strong fluorescence signal compared with that of ADT–CFP. Therefore, *A. tumefaciens* strains containing *ADT–CFP* and p19 constructs (1:1) were infiltrated 1 d before the infiltration of *TP-ssRuBisCO–YFP* and p19 (1:1) constructs. In addition, *A. tumefaciens* containing the *TP-ssRuBisCO–YFP* construct were infiltrated at a lower OD600 of ~0.5. Hence visualization of subcellular localization was performed 4 days post-infiltration (dpi) with *TP-ssRuBisCO–YFP* (which equals 5 dpi, with an *ADT–CFP*).\n",
       "\n",
       "\n",
       "Transient transformants were assayed 5 dpi using a Leica SP2 confocal laser scanning microscope equipped with a ×63 water immersion objective. The abaxial surface of leaf tissue was viewed to observe the localization pattern of fluorescent proteins in the lower epidermis and in mesophyll cells. CFP and chlorophyll were excited with a blue diode laser (405 nm), and emission was collected from 440 nm to 485 nm and from 630 nm to 690 nm, respectively. YFP was excited with a 514 nm argon laser and its emission was collected from 540 nm to 550 nm. For co-localization experiments, CFP and YFP emissions were collected sequentially to avoid emission crosstalk of the fluorophore pair ([Shaner *et al.*, 2005](#CIT0074); [Conley *et al.*, 2009*b*](#CIT0009)). Images were analyzed using Leica Confocal Software (Leica, V2.61) or ImageJ 1.45s ([Schneider *et al.*, 2012](#CIT0073)). Chlorophyll, CFP, and YFP fluorescence was false colored red, cyan, and yellow, respectively.\n",
       "\n",
       "### Western blots\n",
       "\n",
       "\n",
       "For one protein extract, three leaf disc samples (9 mm) were collected from transiently transformed *N. benthamiana* plants at 4 dpi. Total soluble protein (TSP) was extracted and quantified as described by [Conley *et al.* (2009*a*](#CIT0008)). For each sample, 10 μg of TSP were size separated by 10% SDS–PAGE. The proteins were probed with a primary anti-GFP antibody (Clontech catalog no. 632380; designed to recognize GFP and fluorescent variants including CFP) at a 1:5000 dilution. Subsequently, a secondary goat anti-mouse IgG (H+L) horseradish peroxidase-conjugated antibody (Bio-Rad catalog no. 170-6516) was used at a 1:3000 dilution. CFP fusion proteins were visualized with the enhanced chemiluminescence (ECL) detection kit (GE Healthcare, Mississauga, ON, Canada).\n",
       "\n",
       "### Measurements and statistics\n",
       "\n",
       "\n",
       "Stromule and chloroplast lengths were determined using the measuring tool from ImageJ 1.45s ([Schneider *et al.*, 2012](#CIT0073)). Chloroplasts were measured in a straight line across their longest axis. Stromules were considered to be any extensions from chloroplasts that were >1 μm in length. Chloroplasts were analyzed for stromules only if they contained detectable TP-ADT2–CFP fluorescence. For non-linear stromules, several linear measurements were taken to account for bends and curves, and subsequently added together to provide a more accurate measurement of stromule length. Nuclear-localized ADT5–CFP was measured as a proportion of total cells exhibiting ADT5–CFP fluorescence. To determine the proportion of cells having ADT5–CFP within the nucleus, cells were analyzed only if ADT5–CFP was present in the cell/chloroplast. The proportion of chloroplasts with stromules, average stromule and chloroplast lengths, and proportion of cells with ADT5 in the nucleus were analyzed using one-way ANOVA (multiple comparisons) on GraphPad Prism 7.0.\n",
       "\n",
       "## Results\n",
       "\n",
       "\n",
       "### ADTs localize to stroma and stromules of chloroplasts\n",
       "\n",
       "\n",
       "To determine the subcellular localization of ADTs, full-length *A. thaliana* ADTs were transiently expressed as CFP fusions in *N. benthamiana* leaves. *ADT* genes were cloned ([Fig. 1B](#F1)) upstream of the CFP-coding sequence to maintain the fluorescent tag even if the putative TP is cleaved *in planta* ([Li and Chiu, 2010](#CIT0052)). ADTs were tagged with CFP, instead of GFP, to avoid emission spectra overlap with organelle markers tagged with YFP ([Shaner *et al.*, 2005](#CIT0074); [Nelson *et al.*, 2007](#CIT0065); [Lu *et al.*, 2010](#CIT0053)). Leaves of 3- to 5-week-old *N. benthamiana* leaves were co-transformed with an *A. tumefaciens* strain harboring an *ADT–CFP* construct, and a strain encoding p19 to enhance recombinant protein expression ([Voinnet *et al.*, 2003](#CIT0088)). For co-localization experiments, leaves were also co-transformed with *A. tumefaciens* carrying a plasmid-encoded organelle marker. ADT–CFP expression was confirmed by isolating total protein and performing western blots (Supplementary Fig. S2 at *JXB* online).\n",
       "\n",
       "\n",
       "As transformation controls, non-infiltrated tissue, pCB without an insert (empty vector; EV), and pCB infiltrated only with p19 were observed by confocal laser scanning microscopy. For all controls, only chlorophyll autofluorescence was detected and no fluorescence was observed in the CFP and YFP channels in the absence of fusion proteins (Supplementary Fig. S1). ADTs localized to the chloroplast stroma, but were also seen within thread-like structures (e.g. the arrow in ADT2) or as globular structures (e.g. arrows in ADT4) near the chloroplast, but the CFP signal did not directly overlap with chlorophyll autofluorescence ([Fig. 2A](#F2)). The shape and length of these structures were variable, ranging from short and globular to long and narrow protrusions from the chloroplast body. Unlike ADT1–ADT5, ADT6 did not localize to chloroplasts, but was mostly present in the cytosol ([Fig. 2A](#F2), bottom panel).\n",
       "\n",
       "\n",
       "#### Fig. 2.\n",
       "\n",
       "\n",
       "[![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bb7/5444438/c5396de721bc/erx02402.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5444438_erx02402.jpg)\n",
       "\n",
       "\n",
       "\n",
       "[Open in a new tab](figure/F2/)\n",
       "\n",
       "\n",
       "\n",
       "Subcellular localization of ADT–FP fusion proteins and co-localization with TP-ssRuBisCO–YFP. (A) ADT–CFP subcellular localization patterns. ADT1–ADT5 localized to stroma and to areas seemingly close to the chloroplast just outside of the autofluorescence signal generated by chlorophyll. They often appear either in thread-like structures (e.g. the arrow in ADT2) or globular structures (e.g. the arrows in ADT4). The ADT6–CFP pattern is distinctly different, showing a cytosolic distribution. Images were taken at a lower magnification to allow observation of the CFP signal relative to several chloroplasts. (B) Close-ups of ADT–CFP subcellular localization patterns in relation to TP-ssRuBisCO–YFP. In contrast to the chlorophyll autofluorescence, the TP-ssRuBisCO–YFP is a stroma-specific marker that visualizes all stroma-filled areas within the chloroplast including stromules. ADT1–ADT5 are found within the main body of chloroplast and in stromules, while ADT6 is found within the cytosol and does not co-localize with TP-ssRuBisCO–YFP.\n",
       "\n",
       "\n",
       "\n",
       "We hypothesized that the thread-like and globular structures were stromules ([Köhler and Hanson, 2000](#CIT0046)). To confirm that ADTs do localize to stromules, the *ADT–CFP* fusion constructs were co-expressed with the TP of the small subunit of RuBisCO fused to YFP (TP-ssRuBisCO–YFP). This construct is known to localize to the stroma, and therefore can be used to identify stromules ([Nelson *et al.*, 2007](#CIT0065)). The fluorescence of CFP fusion proteins with ADT1–ADT5 overlapped with the fluorescence of TP-ssRuBisCO–YFP, which confirms that these ADTs are targeted to stromules within the chloroplasts ([Fig. 2B](#F2)). The fluorescence of ADT6–CFP did not overlap with the fluorescence of TP-ssRuBisCO–YFP as ADT6 is mostly found within the cytosol ([Fig. 2B](#F2), bottom panel).\n",
       "\n",
       "\n",
       "To determine if the TP domain of ADTs is responsible for chloroplast and stromule localization, the TP sequence was expressed as a TP-ADT2–CFP fusion protein and was detected in chloroplasts and stromules ([Fig. 3](#F3)). These data are consistent with the TP being sufficient to target ADT sequences to chloroplasts and stromules.\n",
       "\n",
       "\n",
       "#### Fig. 3.\n",
       "\n",
       "\n",
       "[![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bb7/5444438/65fe0420402b/erx02403.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5444438_erx02403.jpg)\n",
       "\n",
       "\n",
       "\n",
       "[Open in a new tab](figure/F3/)\n",
       "\n",
       "\n",
       "\n",
       "Localization of ADTs to the chloroplast is dependent on the transit peptide sequences. To test if the transit peptide sequences are sufficient for the transport of ADTs to the chloroplast, the first 99 amino acids of ADT2 (TP-ADT2–CFP) were expressed transiently in *N. benthamiana* leaves. \n",
       "\n",
       "\n",
       "\n",
       "### ADT2 localizes to chloroplasts in a ring structure\n",
       "\n",
       "\n",
       "ADT2–CFP displayed a unique localization pattern compared with the other ADTs ([Fig. 4A](#F4)). In chloroplasts with no apparent central constriction, ADT2–CFP localized to a band at the equatorial plane. Stacked confocal images showed that ADT2 formed a ring around the center of the chloroplast (data not shown). In elongated chloroplasts with a slight indentation, suggestive of an early chloroplast division stage, ADT2–CFP localized as a band around the middle of the elongation exactly at the point of indentation. In chloroplasts with a clear indentation, indicative of a later stage of division, ADT2–CFP was found at the site of constriction. ADT2 localization to the poles of chloroplasts was consistent with remnants of the division ring on daughter chloroplasts ([Miyagishima, 2011](#CIT0060)). These ADT2 localization patterns are strikingly similar to those of proteins that are involved in chloroplast division, a process requiring placement of multiple proteinaceous rings followed by constriction that partitions the chloroplast into two equal-sized daughter chloroplasts ([Fig. 4B](#F4); [Miyagishima, 2011](#CIT0060)).\n",
       "\n",
       "\n",
       "#### Fig. 4.\n",
       "\n",
       "\n",
       "[![Fig. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bb7/5444438/13c3f0004586/erx02404.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5444438_erx02404.jpg)\n",
       "\n",
       "\n",
       "\n",
       "[Open in a new tab](figure/F4/)\n",
       "\n",
       "\n",
       "\n",
       "ADT2 forms structures consistent with chloroplast division rings. (A) In addition to being expressed within chloroplasts and stromules, ADT2 was also found to accumulate in places consistent with chloroplast division rings. The top panel shows ADT2 forming rings at the equatorial plate of the chloroplast. On occasion, ADT2 was found in the constriction zone of chloroplasts (two middle panels). In these cases, the chloroplasts have a distinct dumb-bell shape and the degree of indentation depends on how far the division process has proceeded. In addition, ADT2 accumulated in a spindle-like shape that tapers at chloroplast poles (bottom panel). This fusiform ADT2 accumulation was only found at one pole of the chloroplast and is distinct from a stromule pattern shown in [Fig. 2](#F2). (B) Schematic of chloroplast division stages: from top to bottom, positioning of chloroplast division rings; slightly constricted chloroplast just prior to division; two daughter chloroplast following division. Analogous to the fluorescent images, chloroplasts are shown in red and the position of ring proteins in blue (adapted from [Miyagishima, 2011](#CIT0060)).\n",
       "\n",
       "\n",
       "\n",
       "To initiate division, chloroplasts have to reach a certain size ([Pyke, 1999](#CIT0067)). Therefore, we argue that chloroplasts with ADT2–CFP at the equatorial plane should be the largest as they are in the process of dividing. Conversely, chloroplasts with ADT2–CFP at their pole should be the smallest as they have just recently divided. In addition, these two classes should have little variation in size as they represent distinct stages in chloroplast development. In contrast, growing chloroplasts should vary in size as they encompass all division stages. While chloroplast volume would be the most accurate way to measure chloroplast size, it is difficult to determine. Therefore, chloroplast size was measured as the length of a chloroplast across its longest axis. Chloroplasts from uninfiltrated *N. benthamiana* plants were used to determine the average size of a chloroplast because they should contain chloroplasts at many different developmental stages, and thus different sizes. Chloroplast sizes measured in three uninfiltrated plants had an average length of 5.1 μm ([Table 2](#T2)). In chloroplasts with ADT2–CFP present on a pole, the average length of these chloroplasts was significantly shorter (*P*<0.05), at 4.2 μm ([Table 2](#T2)). Lastly, chloroplasts with ADT2–CFP localized at the equatorial plane were significantly longer, at 6.7 μm (*P*<0.05; [Table 2](#T2)). As predicted, the SD from the mean was larger for chloroplasts from uninfiltrated plants, consistent with a mixed population of chloroplasts, compared with chloroplasts with ADT2–CFP at a pole or at the equatorial plane ([Table 2](#T2)). These data support the hypothesis that the ADT2 localization patterns we observed are consistent with different chloroplast division stages.\n",
       "\n",
       "\n",
       "#### Table 2.\n",
       "\n",
       "\n",
       "\n",
       "Comparison of chloroplast lengths\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "| **Type of chloroplast** | **No. of plants** | **No. of chloroplasts** | **Length (µm**)*a* | **SD** |\n",
       "| --- | --- | --- | --- | --- |\n",
       "| Uninfiltrated | 3 | 68 | 5.1 | 1.12 |\n",
       "| With polar ADT2 | 3 | 75 | 4.2\\* | 0.60 |\n",
       "| With equatorial ADT2 ring | 5 | 35 | 6.7\\* | 0.85 |\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "[Open in a new tab](table/T2/)\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "*a* The length of the chloroplast was measured at its longest axis.\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\\* Significantly different from the chloroplasts in the uninfiltrated control (*P*<0.05) as determined by a *t*-test.\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "### *A single amino acid change in ADT2 affects chloroplast morphology and FtsZ2 localization in* A. thaliana\n",
       "\n",
       "\n",
       "Mutations in genes encoding components of the chloroplast division machinery result in changes to chloroplast morphology ([Pyke, 1999](#CIT0067)). If ADT2 has a role in chloroplast division, it is reasonable to expect that an *adt2* mutant will have distorted chloroplasts. Unlike for other *ADT* genes, no *A. thaliana* T-DNA insertion line that abolishes *ADT2* mRNA production exists ([Corea *et al.*, 2012*b*](#CIT0011)). However, a point mutation within the coding sequence of the ACT regulatory domain of *ADT2* (*adt2-1D*) has been documented in which conversion of a serine to an alanine prevents allosteric inhibition of the enzyme ([Huang *et al.*, 2010](#CIT0034)). To determine if *adt2-1D* affects chloroplast morphology, homozygous *adt2-1D* plants were examined by confocal microscopy and compared with wild-type Col-0 plants of identical age ([Fig. 5A](#F5), [B](#F5)). Chloroplasts in *adt2-1D* plants differed greatly in appearance, and were highly heterogeneous in size and shape ([Fig. 5C](#F5)). This contrasted with chloroplasts in wild-type Col-0, which were ovoid in shape and relatively uniform in size. Although many *adt2-1D* chloroplasts were clearly affected by the mutation, wild-type appearing chloroplasts can still be observed, suggesting a partial loss of ADT2 function.\n",
       "\n",
       "\n",
       "#### Fig. 5.\n",
       "\n",
       "\n",
       "[![Fig. 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bb7/5444438/1d95f0b97079/erx02405.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5444438_erx02405.jpg)\n",
       "\n",
       "\n",
       "\n",
       "[Open in a new tab](figure/F5/)\n",
       "\n",
       "\n",
       "\n",
       "Chloroplast morphology and FtsZ2–YFP localization is affected by a point mutation in *ADT2*. (A) Chloroplasts in wild-type *A. thaliana* Col-0. (B) Chloroplasts in *adt2-1D A. thaliana* mutants. (C) Close-ups of chloroplasts observed in *adt2-1D* to show the heterogeneity in shape and size. (D) Transiently expressed FtsZ–YFP in wild-type Col-0 localizes as expected to a single ring at the equatorial plane. (E) In contrast, FtsZ2–YFP localizes as long spiralling filaments within *adt2-1D* chloroplasts. (D, E) Images of chlorophyll fluorescence (left) and FtsZ2–YFP (middle) are shown separately and merged (right).\n",
       "\n",
       "\n",
       "\n",
       "FtsZ is a tubulin-like protein that is a central component of the chloroplast division apparatus ([Vitha *et al.*, 2001](#CIT0085); [TerBush *et al.*, 2013](#CIT0079)). There are two FtsZ proteins (FtsZ1 and FtsZ2), which assemble to form the earliest known division ring, the Z-ring within the stroma. As the localization of FtsZ in chloroplast division mutants has been used to provide insight into the function of putative division proteins ([Vitha *et al.*, 2003](#CIT0084); [Glynn *et al.*, 2007](#CIT0022); [Fujiwara *et al.*, 2008](#CIT0017); [Glynn *et al.*, 2009](#CIT0023); [Nakanishi *et al.*, 2009*a*](#CIT0061)), we were interested to determine if FtsZ localization was affected in *adt2-1D* plants. Thus we generated an *FtsZ2–YFP* fusion construct and expressed it in *adt2-1D* plants. Expression of FtsZ2–YFP in wild-type Col-0 leaves led to the formation of the expected single ring ([Fig. 5D](#F5)). In contrast, FtsZ2–YFP in *adt2-1D* plants was less organized and formed what appeared to be spirals or multiple rings ([Fig. 5E](#F5)). These results demonstrate that deregulation of ADT2 abolishes proper placement of FtsZ2, further supporting an involvement of ADT2 in chloroplast division.\n",
       "\n",
       "### ADT5 is found in the nucleus\n",
       "\n",
       "\n",
       "In addition to its chloroplast localization, only ADT5–CFP was also detected in nuclei at ~4–5 dpi ([Figs 2A](#F2), [6A](#F6)). To confirm this finding, ADT5–CFP was co-infiltrated with a YFP fusion to the nuclear marker NUP1, a component of the nuclear pore complex in *A. thaliana* that was previously shown to localize to the nuclear membrane ([Lu *et al.*, 2010](#CIT0053)). Confocal imaging determined that NUP1–YFP localized around ADT5–CFP ([Fig. 6B](#F6)). As NUP1–YFP localizes to the nuclear membrane, this result confirms that ADT5–CFP is contained within the nucleus and localizes uniformly throughout the nucleoplasm. As these results were obtained with constructs using a CaMV *35S* promoter, we repeated the experiment and expressed ADT5–CFP under control of its native promoter ([Fig. 6C](#F6)) and confirmed the nuclear localization pattern. To ensure that the observed nuclear localization is not due to a smaller diffusible cleavage product or to a particularly high level of ADT5 compared with other ADTs, we performed western blots ([Fig. 6D](#F6); Supplementary Fig. S2). The blot containing all ADTs expressed under the control of the CaMV *35S* promoter shows a low level of cleavage in all lanes. However, more cleavage product is seen for ADT1 and ADT3 compared with ADT5, while no nuclear localization is observed with either ADT1 or ADT3. The blot showing ADT5–CFP under the control of its native promoter ([Fig. 6D](#F6)) shows a shadow band of the size of CFP, and yet a very clear nuclear localization is evident ([Fig. 6C](#F6)). These results indicate that the nuclear localization pattern is a *bona fide* ADT5 localization. There are additional bands of higher molecular weight ([Fig. 5D](#F5); Supplementary Fig. S2) that might correspond to ADT dimers or even higher multimers.\n",
       "\n",
       "\n",
       "#### Fig. 6.\n",
       "\n",
       "\n",
       "[![Fig. 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bb7/5444438/22e46d96cbf5/erx02406.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5444438_erx02406.jpg)\n",
       "\n",
       "\n",
       "\n",
       "[Open in a new tab](figure/F6/)\n",
       "\n",
       "\n",
       "\n",
       "ADT5 is found in the nucleus. ADT5–CFP proteins are unique as they are the only full-length ADT proteins that were found in the nucleus. (A) Nuclei show a close association with chloroplasts (left) or with stromules of chloroplasts (right). Both images show ADT5–CFP within nuclei. (B) Co-localization of ADT5–CFP with NUP1–YFP. To determine if ADT5–CFP localizes to the nucleus, it was co-expressed with NUP1–YFP in *N. benthamiana*. Images of chlorophyll fluorescence and ADT5–CFP are shown merged (left). NUP1–YFP is shown alone (middle) and merged with ADT5–CFP and chlorophyll fluorescence (right). NUP1–YFP localizes to the nuclear membrane and surrounds ADT5–CFP, confirming that it localizes to the nucleus. (C) ADT5–CFP transiently expressed with its native *ADT5* promoter also localizes to the nucleus. (D) Western blot of ADT5–CFP (calculated size 73.9 kDa) expressed with its native promoter and visualized with a GFP antibody is detected at its expected size. As negative controls, proteins isolated from leaves transformed with GFP (25 kDa) and p19 are shown. Total soluble protein was isolated from transiently transformed leaves, and 10 μg of total soluble protein was size separated by 10% SDS–PAGE. Sizes of the protein ladder are given in kDa.\n",
       "\n",
       "\n",
       "\n",
       "Furthermore, we often observed ADT5–CFP-containing nuclei surrounded by chloroplasts that appeared to be connected to the nucleus through stromules ([Fig. 6A](#F6)), suggesting that ADT5 nuclear localization depends on stromule-mediated transport. To test if stromule formation affects the nuclear localization of ADT5, we transiently expressed myosin XI tail domains (*dnMyoXI-2* and *dnMyoXI-K/GTD*) as they were previously shown to inhibit stromule formation through a dominant negative effect on wild-type myosin XI ([Avisar *et al.*, 2008](#CIT0002); [Natesan *et al.*, 2009](#CIT0064)). To ensure that these treatments inhibit stromules, several control transformations were performed. To visualize stromules in control transformations, TP-ADT2–CFP was used, as it easily visualizes stromules and therefore provides a very sensitive marker to detect stromule inhibition. TP-ADT2–CFP was co-infiltrated into *N. benthamiana* leaves with an empty pCB vector, a *dnMyoXI-2* construct, or a *dnMyoXI-K/GTD* construct ([Avisar *et al.*, 2008](#CIT0002)), and the number of chloroplasts having stromules ([Fig. 7A](#F7)) and the length of stromules formed ([Fig. 7B](#F7)) were determined. In plants transformed with an empty vector, 28.7% of chloroplasts had stromules ([Fig. 7A](#F7)) and the average length of these was 4.6 μm ([Fig. 7B](#F7)). Infiltration with dnMyoXI-2 decreased the percentage of chloroplasts with stromules (22.2%; [Fig. 7A](#F7)) compared with that of the control, and significantly reduced (*P*<0.001) the average length of stromules to 2.7 μm ([Fig. 7B](#F7)). Treatment with dnMyoXI-K/GTD also caused a significant decrease (*P*<0.001) in the percentage of chloroplasts with stromules (17.3%; [Fig. 7A](#F7)) and significantly decreased (*P*<0.001) the average length of stromules to 3.5 μm ([Fig. 7B](#F7)). These results confirm that both myosin domains affect stromule formation.\n",
       "\n",
       "\n",
       "#### Fig. 7.\n",
       "\n",
       "\n",
       "[![Fig. 7.\n",
       "](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bb7/5444438/d14c09b73745/erx02407.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5444438_erx02407.jpg)\n",
       "\n",
       "\n",
       "\n",
       "[Open in a new tab](figure/F7/)\n",
       "\n",
       "\n",
       "\n",
       "The presence of ADT5 in the nucleus is affected by the ability to form stromules. To determine if nuclear localization of ADT5 is dependent on stromules, plants were co-infiltrated with TP-ADT2–CFP (A and B) as a control or ADT5–CFP (C) and an empty vector (dark gray), dominant negative myosin XI-2 (dnMyoXI-2; light gray) and myosin XI-K (dnMyoXI-K/GTD; white), respectively. (A) Percentage of chloroplasts having stromules. Chloroplasts were analyzed if they contained any visible TP-ADT2–CFP fluorescence and were determined to have a stromule if the projection was longer than 1 μm. In total 554, 395, and 579 chloroplasts were analyzed from plants transformed with an empty vector, dnMyoXI-2, and dnMyoXI-K/GTD, respectively. (B) Average length of stromules. A total of 166, 93, and 91 stromules were measured from plants transformed with an empty vector, dnMyoXI-2, and dnMyoXI-K/GTD, respectively. (C) Nuclear localization of ADT5–CFP. Cells were analyzed for CFP fluorescence in the nucleus only if any ADT5–CFP fluorescence was detectable. A total of 131, 190, and 358 cells were analyzed from plants transformed with an empty vector, dnMyoXI-2, and dnMyoXI-K/GTD, respectively. Each experiment was performed on three independent occasions. Significant differences (*P*<0.001) as determined by a one-way ANOVA (multiple comparisons) are indicated by different letters. Averages ± SE of the mean are plotted.\n",
       "\n",
       "\n",
       "\n",
       "To determine if the ability to form stromules affects nuclear localization of ADT5, *ADT5–CFP* was co-expressed with the empty pCB vector, *dnMyoXI-2*, or *dnMyoXI-K/GTD* ([Fig. 7C](#F7)). The extent of ADT5–CFP nuclear localization was expressed as a percentage of cells containing CFP fluorescence. Co-infiltration with the empty vector control showed that 25.9% of cells had ADT5–CFP fluorescence visible in the nucleus ([Fig. 7C](#F7)). In contrast, co-infiltration with *dnMyoXI-2* or *dnMyoXI-K/GTD* showed that ADT5–CFP was detected in the nucleus in only 7.1% and 4.3% of cells, respectively ([Fig. 7C](#F7)), both significant reductions (*P*<0.001) from the control. These data demonstrate that ADT5–CFP nuclear localization is decreased by the same conditions shown to decrease stromule formation.\n",
       "\n",
       "### *ADTs in* A. thaliana\n",
       "\n",
       "\n",
       "Transient transformations using agroinfiltration are widely used in *N. benthamiana* but have traditionally been difficult in *A. thaliana* ([Wroblewski *et al.*, 2005](#CIT0091)). We were able to transform *A. thaliana* reliably by growing plants with 20 mM l-ascorbic acid, which seemed to decrease necrosis of leaves associated with agroinfiltration. As it was not the focus of this study, the reason for this was not addressed. However, as l-ascorbic acid can scavenge damaging reactive oxygen species ([Gallie, 2013](#CIT0019)), the increased tolerance of *A. thaliana* to agroinfiltration may be due to a decrease in oxidative stress.\n",
       "\n",
       "\n",
       "\n",
       "*ADT–CFP* fusion genes were transiently expressed in *A. thaliana* to confirm that localization in a heterologous host reflected the situation in the native environment ([Fig. 8](#F8)). As in *N. benthamiana*, all ADT–CFPs in *A. thaliana*, with the exception of ADT6–CFP, which appeared in the cytosol, exhibited stroma and stromule-like patterns ([Fig. 8A](#F8)). Similarly, the unique localization patterns of ADT2–CFP and ADT5–CFP were observed in *A. thaliana*, localizing to the equatorial plane and poles of chloroplasts ([Fig. 8B](#F8)), and to nuclei ([Fig. 8C](#F8)), respectively. This finding was important as it verifies that the findings in *N. benthamiana* are not artifacts.\n",
       "\n",
       "\n",
       "#### Fig. 8.\n",
       "\n",
       "\n",
       "[![Fig. 8.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bb7/5444438/64e45024f128/erx02408.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5444438_erx02408.jpg)\n",
       "\n",
       "\n",
       "\n",
       "[Open in a new tab](figure/F8/)\n",
       "\n",
       "\n",
       "\n",
       "ADT localization to the stroma and stromules, the chloroplast equatorial plane, and the nucleus can also be detected in *A. thaliana*. To test if the ADT patterns determined in *N. benthamiana* reflect expression in *A. thaliana*, all six ADT–CFP fusion proteins were transiently expressed in *A. thaliana* Col-0. All images show a merge of chlorophyll and CFP fluorescence. (A) ADT1–CFP through ADT5–CFP localize to stroma and structures resembling stromules of varying shapes and lengths, with varying levels of fluorescence in the stroma. ADT6–CFP localizes outside of chloroplasts in the cytosol. (B) Chloroplast division patterns for ADT2–CFP. (C) Nuclear localization of ADT5–CFP.\n",
       "\n",
       "\n",
       "\n",
       "## Discussion\n",
       "\n",
       "\n",
       "### Phenylalanine biosynthesis and stromules\n",
       "\n",
       "\n",
       "ADTs were seen to localize to thread-like structures seemingly outside of the region of chlorophyll autofluorescence ([Fig. 2A](#F2)). Co-localization of ADT–CFP fluorescence with the stromule marker TP-ssRuBisCO–YFP ([Nelson *et al.*, 2007](#CIT0065)) confirmed that all ADTs except ADT6 localized to stromules ([Fig. 2B](#F2)). Prior to our study, *A. thaliana* ADTs were found uniformly throughout the stroma of chloroplasts, with no indication of stromule localization ([Rippert *et al.*, 2009](#CIT0071)). However, the said study used protoplasts, which represent dedifferentiated cells that are also in a state of stress ([Genschik *et al.*, 1992](#CIT0021); [Reyes *et al.*, 2010](#CIT0070)).\n",
       "\n",
       "\n",
       "Stromules are dynamic structures that range in size and shape from short beak-like projections to long and elaborate tubules ([Köhler and Hanson, 2000](#CIT0046); [Gunning, 2005](#CIT0026); [Gray, 2013](#CIT0024)). The function of stromules has been the subject of debate, but the idea that they increase transport of compounds synthesized within plastids to other areas of the cell is now generally accepted ([Hanson and Sattarzadeh, 2013](#CIT0028)). The presence of biosynthetic enzymes, such as ADTs, in stromules is consistent with this hypothesis. Phenylalanine, the product of ADT enzymatic activity, is required within the cytosol for the synthesis of proteins and as a precursor for phenylpropanoids such as lignins and flavonoids ([Fraser and Chapple, 2011](#CIT0016)). It stands to reason that stromules could have a high concentration of phenylalanine as a result of ADT activity, providing an effective means of increasing phenylalanine export into the cytosol. Interestingly, abiotic stressors, such as drought and salt stress, known to induce stromules ([Gray *et al.*, 2012](#CIT0025)), are also associated with increased flavonoid levels in leaves ([Agati *et al.*, 2011](#CIT0001); [Mewis *et al.*, 2012](#CIT0059)), and genes encoding flavonoid biosynthetic enzymes are up-regulated in response to salinity-induced stress ([Walia *et al.*, 2005](#CIT0089)). For example, PHENYLALANINE AMMONIA LYASE1 (PAL1) uses phenylalanine as a substrate to catalyze the first step of phenylpropanoid biosynthesis in the cytosol ([Walia *et al.*, 2005](#CIT0089); [Fraser and Chapple, 2011](#CIT0016)). Under conditions of salt stress, *PAL1* up-regulation coincides with the formation of stromules, suggesting that these two events might be linked and that both events contribute to an increased transport of phenylalanine to the cytosol.\n",
       "\n",
       "\n",
       "ADTs are not the only enzymes that have been associated with stromules. Other examples include geranylgeranyl diphosphate synthase (GGPS), which synthesizes geranylgeranyl diphosphate in the stromules of chloroplasts, and the compound is required in the cytosol as part of isoprenoid metabolism ([Thabet *et al.*, 2012](#CIT0080)), or RuBisCO and an aspartate aminotransferase (ASP5), which are present in stromules and capable of moving between plastids via stromules when expressed as GFP fusion proteins ([Kwok and Hanson, 2004](#CIT0050)). This allows for the speculation that chloroplastic enzymes that synthesize molecules required in the cytosol preferentially localize to stromules as these might facilitate metabolite export.\n",
       "\n",
       "\n",
       "\n",
       "*In silico* analysis of *A. thaliana* ADT sequences using ChloroP ([Emanuelsson *et al.*, 1999](#CIT0015)) predicted that their N-terminal sequences were likely to encode TPs directing the enzymes to the chloroplast. We present data that confirm the *in silico* prediction for ADT2, as the N-terminal portion was necessary and sufficient to allow direct import into chloroplasts and specifically stromules ([Fig. 3A](#F3), [B](#F3)). These data also corroborate observations made for all three petunia ADTs, where the N-terminal portion of ADTs directed GFP fusion proteins to chloroplasts ([Maeda *et al.*, 2010](#CIT0056)).\n",
       "\n",
       "### ADT2 and a role in chloroplast division\n",
       "\n",
       "\n",
       "Aside from its inclusion in stroma and stromules, ADT2 localized as a ring around the equatorial plane or at the poles of chloroplasts ([Figs 4A](#F4), [8B](#F8)). The similarity of these patterns to those of chloroplast division proteins ([Miyagishima, 2011](#CIT0060)) during and after division led to an investigation of a possible second, non-enzymatic role for ADT2 in chloroplast division. Since chloroplast division is regulated by size ([Pyke, 1999](#CIT0067)), we reasoned that chloroplasts with either the equatorial or polar localization patterns would be larger and smaller, respectively, than average sized chloroplasts. This was confirmed upon comparing chloroplast lengths across their longest axis ([Table 2](#T2)). Additionally, the SD of chloroplast lengths varied. It was lowest in chloroplasts with ADT2 at the equator or at a pole, in agreement with these chloroplasts being in very distinct phases, either just prior to or post-division, and therefore very similar in size. This was in contrast to chloroplasts from uninfiltrated plants, which are comprised of chloroplasts in all division states. Therefore, our results suggest that ADT2 localizes to the division plane early in the division process and remains there throughout the duration of constriction and separation into daughter organelles. Similarly, known division proteins such as FtsZ and ACCUMULATION AND REPLICATION OF CHLOROPLASTS 6 (ARC6) assemble at the equatorial plane in the process leading to constriction and division ([Vitha *et al.*, 2001](#CIT0085); [2003](#CIT0084)).\n",
       "\n",
       "\n",
       "The striking similarities between ADT2 and other chloroplast division proteins prompted observation of chloroplasts in *adt2* mutant plants. Interestingly, no T-DNA insertion knockout lines that abolish *ADT2* mRNA are available ([Corea *et al.*, 2012*b*](#CIT0011)). This makes ADT2 unique and raises the possibility that an *adt2* knockout is lethal. However, plants homozygous for the *adt2-1D* point mutation have been documented ([Huang *et al.*, 2010](#CIT0034)). The appearance of *adt2-1D* chloroplasts was variable and the presence of misshapen and heterogeneous chloroplasts is consistent with previous descriptions of chloroplast morphology in division mutants ([Pyke and Leech, 1992](#CIT0068); [Colletti *et al.*, 2000](#CIT0007); [Glynn *et al.*, 2009](#CIT0023); [Nakanishi *et al.*, 2009*b*](#CIT0062)). In the *adt2-1D* plants, chloroplasts that appear wild type can still be observed. This infers that the single amino acid substitution probably does not abolish ADT2’s function, but impairs it. During chloroplast division, FtsZ forms the first known division ring within the stroma ([TerBush *et al.*, 2013](#CIT0079)). Given the central role that FtsZ proteins play in division, we reasoned that FtsZ localization should be affected in *adt2-1D* chloroplasts if ADT2 is a chloroplast division protein. Expression of an *FtsZ2–YFP* fusion construct in *adt2-1D* plants revealed long and spiraling FtsZ2–YFP filaments throughout the chloroplast stroma. Although FtsZ2–YFP was overexpressed in our study, in wild-type Col-0 chloroplasts the fusion protein localized as expected as a single equatorial ring. We propose that the abnormal appearance of FtsZ2–YFP within *adt2-1D* chloroplasts suggests that ADT2 regulates FtsZ positioning. However, we cannot ignore the possibility that elevated phenylalanine levels in *adt2-1D* (up to 160-fold compared with the wild type; [Huang *et al.*, 2010](#CIT0034)) are at least in part responsible for the observed changes.\n",
       "\n",
       "\n",
       "It is intriguing to note that not all ring proteins have been identified. Although FtsZ, one of the inner rings, and ARC5 (dynamin), one of the outer rings, have been known for some time ([Vitha *et al.*, 2001](#CIT0085); [Gao *et al.*, 2003](#CIT0020)), the identity of the outer plastid-dividing (PD) ring ([Yoshida *et al.*, 2010](#CIT0095)), polyglucan filaments, has just recently been suggested, and the composition of the inner PD ring is still unknown. The realization that enzymes can have a second unrelated, non-enzymatic function, even as part of cellular structural components, is discovered more and more frequently ([Huberts and van der Klei, 2010](#CIT0035); MoonProt Database: [www.moonlightingproteins.org, last accessed 13 February 2017](http://www.moonlightingproteins.org)). A good example of this is the *Physcomitrella patens* enzyme presenilin, the catalytic unit for γ-secretase, which has an independent function in the cytoskeletal network ([Khandelwal *et al.*, 2007](#CIT0043)). However, additional studies will be required to determine the precise relationship between ADT2, phenylalanine levels, chloroplast division, and other components of the chloroplast division machinery.\n",
       "\n",
       "### ADT5 and the nucleus\n",
       "\n",
       "\n",
       "Similar to ADT2, ADT5 has an additional unique localization pattern and was clearly observed in the nuclei of both *N. benthamiana* and *A. thaliana* ([Fig. 6A](#F6), [C](#F6)). Proteins with dual plastid and nuclear localization may be significant in the context of retrograde signaling. While retrograde signaling traditionally refers to chemical messengers that are released from plastids and affect nuclear gene expression ([Inaba *et al.*, 2011](#CIT0036)), it is becoming apparent that proteins within the chloroplast can also act as retrograde signals ([Isemer *et al.*, 2012](#CIT0037); [Krause *et al.*, 2012](#CIT0049)). One such protein is WHIRLY1 from *A. thaliana*, which can move directly from plastids to the nucleus ([Isemer *et al.*, 2012](#CIT0037)). In plastids, WHIRLY1 contributes to plastid genome stability by preventing illegitimate recombination ([Maréchal *et al.*, 2009](#CIT0058)). In the nucleus, it acts as a transcriptional activator of pathogen response genes ([Isemer *et al.*, 2012](#CIT0037)), consistent with the increased pathogen susceptibility associated with decreased WHIRLY1 DNA binding ability ([Desveaux *et al.*, 2005](#CIT0012)). Whether ADT5 has a role in retrograde signaling is currently unknown. In addition, many other enzymes with diverse functions have been reported in both plastids and nuclei, such as phosphate-isopentyltransferase 3, an enzyme involved in cytokinin biosynthesis ([Galichet *et al.*, 2008](#CIT0018); [Krause *et al.*, 2012](#CIT0049)), CDT1, a kinase required in cell cycle regulation ([Raynaud *et al.*, 2005](#CIT0069)), and a dihydrofolate reductase required for nucleotide metabolism ([Luo *et al.*, 1997](#CIT0054)), but often the nuclear role of these enzymes is ill defined.\n",
       "\n",
       "\n",
       "While a direct mechanism of protein transport between plastids and the nucleus through stromules is a hypothetical mode ([Krause *et al.*, 2012](#CIT0049)), they have been shown to interconnect plastids ([Köhler *et al.*, 1997](#CIT0045); [Kwok and Hanson, 2004](#CIT0050); [Hanson and Sattarzadeh, 2013](#CIT0028)). Chloroplasts with stromules containing ADT5–CFP often appeared to connect directly with the nucleus ([Fig. 6A](#F6)). Expression of dominant negative forms of myosin XI were found to inhibit stromules ([Fig. 7A](#F7), [B](#F7)) and also significantly reduce ADT5–CFP localization to the nucleus ([Fig. 7C](#F7)), providing indirect evidence of stromule-mediated nuclear transport. We are aware that there are alternative interpretations of these results, as myosin XI is involved in other processes including movement of organelles ([Avisar *et al.*, 2008](#CIT0002)) and cytoplasmic streaming ([Shimmen and Yokota, 2004](#CIT0075)). Regardless of this, the appearance of stromules directly connecting to nuclei ([Fig. 6A](#F6)) makes the possibility of a stromule-mediated nuclear transport system intriguing.\n",
       "\n",
       "\n",
       "Currently, the role that ADT5 plays in the nucleus is unknown. It is conceivable that it acts as a transcriptional regulator of *ADT* genes or other genes within the same biosynthetic pathway. There is precedence for an enzyme to act as a transcriptional regulator of functionally related genes. For example, *A. thaliana* HEXOKINASE1 (HXK1) is involved in glucose metabolism in mitochondria, but also localizes to the nucleus where it forms part of a protein complex affecting transcription of genes involved in glucose signaling ([Cho *et al.*, 2006](#CIT0006)). Furthermore, in the budding yeast *Saccharomyces cerevisiae*, ARG5,6, an enzyme involved in arginine biosynthesis, is able to bind DNA directly and regulate gene expression ([Hall *et al.*, 2004](#CIT0027)). Interestingly, mutant analysis has shown that loss of ADT5 activity cannot be compensated for by other ADTs and is the only single *ADT* knockout with a visible phenotype ([Corea *et al.*, 2012*b*](#CIT0011)), consistent with a unique nuclear role for ADT5.\n",
       "\n",
       "### ADT2 and ADT5: moonlighting proteins\n",
       "\n",
       "\n",
       "The term ‘moonlighting protein’ was coined to describe proteins that perform multiple autonomous and often unrelated functions without these functions being partitioned into different domains of the protein or resulting from alternative splicing or gene fusion (reviewed in [Jeffery, 1999](#CIT0039), [2013](#CIT0040)). Since the description of the first moonlighting proteins, the number of documented moonlighting proteins has increased to ~300 ([Jeffery, 2013](#CIT0040); MoonProt Database: [www.moonlightingproteins.org](http://www.moonlightingproteins.org)). Enzymes are common among moonlighting proteins and many have additional non-enzymatic functions, including roles as structural components and as regulators of transcription or translation ([Jeffery, 2013](#CIT0040)). For example, in *Tetrahymena*, a citrate synthase acts as an enzyme in mitochondria while in the cytosol it can polymerize to form 14 nm filaments and then act as a cytoskeletal protein ([Kojima *et al.*, 1997](#CIT0047)). We propose that ADT2 and ADT5 are moonlighting proteins. In the current study we provide evidence that ADT2, with demonstrated arogenate dehydratase activity ([Cho *et al.*, 2007](#CIT0005)), can form rings around chloroplasts similar to FtsZ or ARC5 as part of their role in chloroplast division ([Vitha *et al.*, 2001](#CIT0085); [Gao *et al.*, 2003](#CIT0020)). The dual localization of ADT5 to chloroplasts and nuclei suggests that ADT5 has an additional role in the nucleus, possibly in transcriptional regulation. As the entire ADT enzyme family catalyzes the final step in phenylalanine biosynthesis, we expect this to be a key regulatory step.\n",
       "\n",
       "\n",
       "Moonlighting proteins appear to be ubiquitous in nature, with documented examples in simple single-cell organisms, such as archaea and bacteria, and complex eukaryotes including plants and animals. It seems likely that moonlighting proteins are evolutionarily advantageous, ensuring that the number of genes in a genome does not limit the number of functions they are capable of performing. Many moonlighting proteins are ancient in terms of their evolutionary history, giving them ample time to adapt to a second role ([Jeffery, 2013](#CIT0040)). Analyzing the subcellular localization of *A. thaliana* ADTs shows that the function of enzymes is far more complex than previously realized.\n",
       "\n",
       "## Supplementary data\n",
       "\n",
       "\n",
       "Supplementary data are available at *JXB* online.\n",
       "\n",
       "\n",
       "Fig. S1. Negative controls.\n",
       "\n",
       "\n",
       "Fig. S2. Western blots showing expression of transiently expressed ADTs.\n",
       "\n",
       "## Supplementary Material\n",
       "\n",
       "\n",
       "\n",
       "Supplementary\\_Figure1\\_2\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "[Click here for additional data file.](/articles/instance/5444438/bin/erx024_suppl_Supplementary_Figure1_2.pdf) (252.8KB, pdf) \n",
       "\n",
       "\n",
       "## Acknowledgments\n",
       "\n",
       "\n",
       "We would like to thank Lisa Amyot and Reza Saberianfar for their technical support when using the confocal microscope, and Drs Gary Tian and Yuhai Cui, for the donation of the NUCLEOPORIN1–YFP nuclear marker. We are grateful to Agriculture and Agri-Food Canada (AAFC) in London for providing generous access to their confocal microscope. *Nicotiana benthamiana* seeds and *Agrobacterium* strains were provided by Jamie McNeil and Dr R. Menassa, AAFC, London, Ontario. Many thanks to Dr Tengfang Huang and Dr Georg Jander (Boyce Thompson Institute for Plant Research (Ithaca, NY) for generously providing us with *adt2-1D* seeds. We would like to express our thanks to Drs R. Menassa and K. Dobinson, AAFC, London, Ontario, Emily Clayton, and Emily Cornelius for diligently proofreading the manuscript. This work was supported in part by a grant from MITACS to CDB and SEK, and an NSERC Discovery grant to SEK.\n",
       "\n",
       "## Glossary\n",
       "\n",
       "\n",
       "### Abbreviations:\n",
       "\n",
       "\n",
       "\n",
       "ACT\n",
       "aspartokinase–chorismate mutase–TyrA\n",
       "\n",
       "\n",
       "ADT\n",
       "arogenate dehydratase\n",
       "\n",
       "\n",
       "CFP\n",
       "cyan fluorescent protein\n",
       "\n",
       "\n",
       "dpi\n",
       "days post-infiltration\n",
       "\n",
       "\n",
       "EV\n",
       "empty vector\n",
       "\n",
       "\n",
       "GFP\n",
       "green fluorescent protein\n",
       "\n",
       "\n",
       "FtsZ\n",
       "filamentous temperature sensitive Z\n",
       "\n",
       "\n",
       "PAT\n",
       "prephenate aminotransferase\n",
       "\n",
       "\n",
       "PDT\n",
       "prephenate dehydratase\n",
       "\n",
       "\n",
       " PP\n",
       "phenylpyruvate\n",
       "\n",
       "\n",
       "PPAT\n",
       "phenylpyruvate aminotransferase\n",
       "\n",
       "\n",
       "PTGS\n",
       "post-transcriptional gene silencing\n",
       "\n",
       "\n",
       "TP\n",
       "transit peptide\n",
       "\n",
       "\n",
       "YFP\n",
       "yellow fluorescent protein.\n",
       "\n",
       "\n",
       "## Associated Data\n",
       "\n",
       "\n",
       "*This section collects any data citations, data availability statements, or supplementary materials included in this article.*\n",
       "\n",
       "\n",
       "### Supplementary Materials\n",
       "\n",
       "\n",
       "\n",
       "Supplementary\\_Figure1\\_2\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "[Click here for additional data file.](/articles/instance/5444438/bin/erx024_suppl_Supplementary_Figure1_2.pdf) (252.8KB, pdf) \n",
       "\n",
       "\n",
       "\n",
       "\n",
       "---\n",
       "\n",
       "\n",
       "Articles from Journal of Experimental Botany are provided here courtesy of **Oxford University Press**\n",
       "\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from scifeeder.ncbi import NCBI\n",
    "\n",
    "x = NCBI.from_pmcid(p.pmcid)\n",
    "\n",
    "markdown = x.article()\n",
    "Markdown(markdown)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ab593284-e993-47e6-ab7d-1fb50509f5eb",
   "metadata": {},
   "source": [
    "----\n",
    "\n",
    "## Runner"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b640e0cf-7bde-422c-b6b7-6ef880d0bb4c",
   "metadata": {},
   "outputs": [],
   "source": [
    "from scifeeder.runner import Runner\n",
    "from scifeeder.cache import Cache\n",
    "from scifeeder.epmc import EPMC\n",
    "from scifeeder.ncbi import NCBI"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6d7a0eff-acff-4e11-bb4e-5a99c03c1d73",
   "metadata": {},
   "outputs": [],
   "source": [
    "from scifeeder.utils import read_papers_csv\n",
    "from scifeeder.types import Paper\n",
    "papers=list(read_papers_csv('suba_j.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b03d47d4-20f7-42d3-b9d4-5ed413d6d0b0",
   "metadata": {},
   "outputs": [],
   "source": [
    "DONE = {'11511543', '20847098', '21051552'}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7bbb86f2-d473-40b6-9c7b-902448f12e04",
   "metadata": {},
   "outputs": [],
   "source": [
    "for p in papers:\n",
    "    if p.pmcid and p.pmid not in DONE and p.year > 2010:\n",
    "        break\n",
    "p"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4755a329-081b-43e3-8c71-278b1a96562c",
   "metadata": {},
   "outputs": [],
   "source": [
    "len([p for p in papers if p.pmcid]), len(papers), len([p for p in papers if p.doi])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "deac2b48-efa6-4f3e-8e29-b2820e1d77ab",
   "metadata": {},
   "outputs": [],
   "source": [
    "what = [p for p in papers if not p.doi and not p.pmcid]\n",
    "\n",
    "for p in what:\n",
    "    url=f\"https://ncbi.nlm.nih.gov/pubmed/{p.pmid}/\"\n",
    "    print(p.pmid, p.year, url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0246036c-c1bc-4339-a789-810e976ecea7",
   "metadata": {},
   "outputs": [],
   "source": [
    "html = x.html(True)\n",
    "with open('saved.html', 'wt') as fp:\n",
    "    fp.write(html)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ef94dfdd-a61e-4ed3-b96e-bda5eb05ad65",
   "metadata": {},
   "outputs": [],
   "source": [
    "cache = Cache('mycache')\n",
    "cache.fetch_ncbi(p)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f02cf53c-1768-42a9-941d-0ec2965cc7d1",
   "metadata": {},
   "outputs": [],
   "source": [
    "from scifeeder.runner import Runner\n",
    "from scifeeder.selenium_cls import Selenium, StealthSelenium\n",
    "from scifeeder.cache import Cache\n",
    "\n",
    "from scifeeder.issn import ISSN_MAP\n",
    "class SeleniumRunner(Runner):\n",
    "\n",
    "    def start(self):\n",
    "\n",
    "        self.cache = Cache('scache')\n",
    "        self.web = self.create_driver()\n",
    "        self.ok = 0\n",
    "        self.failed = 0\n",
    "\n",
    "    def create_driver(self):\n",
    "        return StealthSelenium(headless=True)\n",
    "        \n",
    "    def work(self, paper, tqdm):\n",
    "        tqdm.write(f'working... {paper.pmid}')\n",
    "        if not paper.doi:\n",
    "            return 'nodoi'\n",
    "        if paper.issn not in ISSN_MAP:\n",
    "            return 'noissn'\n",
    "\n",
    "        try:\n",
    "            html =  self.web.fetch_html(paper.doi,ISSN_MAP[paper.issn])\n",
    "            if html is None:\n",
    "                self.web = self.create_driver()\n",
    "                tqdm.write('retry....')\n",
    "                html =  self.web.fetch_html(paper.doi,ISSN_MAP[paper.issn])\n",
    "            if html is None:\n",
    "                retval= 'cc'\n",
    "            elif not html:\n",
    "                retval= 'timeout'\n",
    "            else:\n",
    "                self.cache.save_html(paper, html)\n",
    "                retval = 'ok'\n",
    "        except Exception as e:\n",
    "            tqdm.write(f'failed: {paper.pmid} {e}')\n",
    "            retval= 'failed'\n",
    "        tqdm.write(retval)\n",
    "        return retval\n",
    "\n",
    "    def end(self):\n",
    "        self.web.close()\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b7f6f190-a68f-4321-8dc0-da8895e9b593",
   "metadata": {},
   "outputs": [],
   "source": [
    "runner = SeleniumRunner('suba_j.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6d7bfd25-b1fe-4e53-b6bf-b3e5f49a5556",
   "metadata": {},
   "outputs": [],
   "source": [
    "runner.run(True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "cf09e260-d04d-48a2-899f-e80ebd89e840",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
